<!DOCTYPE HTML>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>Sequence - FHIR v1.7.0</title>

  
  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
  <meta name="author" content="http://hl7.org/fhir"/>

  <link rel="stylesheet" href="fhir.css"/>
  <link rel="Prev" href="http://hl7.org/fhir/sequence-definitions.html"/>

    <!-- Bootstrap core CSS -->
  <link rel="stylesheet" href="./dist/css/bootstrap.css"/>
  <link rel="stylesheet" href="./assets/css/bootstrap-fhir.css"/>

    <!-- Project extras -->
  <link rel="stylesheet" href="./assets/css/project.css"/>
  <link rel="stylesheet" href="./assets/css/pygments-manni.css"/>
	<link rel="stylesheet" href="jquery-ui.css"/>

    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- [if lt IE 9]>
  <script src="./assets/js/html5shiv.js"></script>
  <script src="./assets/js/respond.min.js"></script>
  <![endif] -->

    <!-- Favicons -->
  <link sizes="144x144" rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-144-precomposed.png"/>
  <link sizes="114x114" rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-114-precomposed.png"/>
  <link sizes="72x72" rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-72-precomposed.png"/>
  <link rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-57-precomposed.png"/>
  <link rel="shortcut icon" href="./assets/ico/favicon.png"/>

</head>
<body>
	<div id="segment-header" class="segment">  <!-- segment-header -->	
		<div class="container">  <!-- container -->
			<a no-external="true" id="logo" href="http://hl7.org/fhir"><img src="./assets/images/fhir-logo-www.png" alt="logo fhir"/> </a>
			<div id="hl7-status">
				<b>Current Build</b>
			</div>       	
			
			<div id="hl7-nav">
				 <a no-external="true" id="hl7-logo" href="http://www.hl7.org">
					<img src="./assets/images/hl7-logo.png" width="42" alt="visit the hl7 website" height="50"/>
				</a> 
			</div>       	
			<div id="hl7-nav"><a no-external="true" id="hl7-logo" href="http://hl7.org/fhir/search.cfm"><img src="./assets/images/search.png" alt="Search FHIR"/></a></div> 
		</div>
		<div class="container">  <!-- container -->
	</div></div>  <!-- /segment-header -->	


	<div id="segment-navbar" class="segment">  <!-- segment-navbar -->
		<div id="stripe"> </div>
		<div class="container">  <!-- container -->
   <!-- HEADER CONTENT -->
 	      		 	      		      		
			<nav class="navbar navbar-inverse">
				<div class="container">
					<button data-target=".navbar-inverse-collapse" data-toggle="collapse" type="button" class="navbar-toggle">
						<span class="icon-bar"> </span>
						<span class="icon-bar"> </span>
						<span class="icon-bar"> </span>
					</button>
					<a href="index.html" class="navbar-brand hidden">FHIR</a>
					<div class="nav-collapse collapse navbar-inverse-collapse">
						<ul class="nav navbar-nav">
              <li><a href="./index.html">Home</a></li>
              <li><a href="./modules.html">Getting Started</a></li>
              <li><a href="./documentation.html">Documentation</a></li>
              <li><a href="./resourcelist.html">Resources</a></li>
              <li><a href="./profilelist.html">Profiles</a></li>
              <li><a href="./extensibility-registry.html">Extensions</a></li>
              <li><a href="./operationslist.html">Operations</a></li>
              <li><a href="./servicelist.html">Services</a></li>
						</ul>
					</div>  <!-- /.nav-collapse -->
				</div>  <!-- /.container -->
			</nav>  <!-- /.navbar -->
      		
  <!-- /HEADER CONTENT -->				
		</div>  <!-- /container -->
	</div>  <!-- /segment-navbar -->
	

	<div id="segment-breadcrumb" class="segment">  <!-- segment-breadcrumb -->
		<div class="container">  <!-- container -->	
			<ul class="breadcrumb">
        <li><a href="diagnostics-module.html"><img src="diagnostics.jpg"/> Diagnostics</a></li>        <li><a href="sequence.html">Sequence</a></li>        <li><b>Detailed Descriptions</b></li>          <!-- sequence.html / res-Detailed Descriptions / Sequence -->

			</ul>	
		</div>  <!-- /container -->
	</div>  <!-- /segment-breadcrumb -->
	

	<div id="segment-content" class="segment">  <!-- segment-content -->
	<div class="container">  <!-- container -->
            <div class="row">
            	<div class="inner-wrapper">
  <!-- CONTENT CONTENT -->


<div class="col-9">

<ul class="nav nav-tabs"><li><a href="sequence.html">Content</a></li><li><a href="sequence-examples.html">Examples</a></li><li class="active"><a href="#">Detailed Descriptions</a></li><li><a href="sequence-mappings.html">Mappings</a></li><li><a href="sequence-profiles.html">Profiles &amp; Extensions</a></li></ul>

<h2 class="self-link-parent"><span class="sectioncount">10.4.6<a name="10.4.6"> </a></span> Resource Sequence - Detailed Descriptions <a href="sequence-definitions.html#10.4.6" title="link to here" class="self-link"><img src="assets/images/link.svg" width="20" class="self-link" height="20"/></a></h2>

<p>Detailed Descriptions for the elements in the Sequence resource.</p>

<table class="dict">
  <tr><td colspan="2" class="structure"><a name="Sequence"> </a><b>Sequence</b></td></tr>
  <tr><td>Definition</td><td><p>Raw data describing a biological sequence.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>1..1</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.identifier"> </a><b>Sequence.identifier</b></td></tr>
  <tr><td>Definition</td><td><p>A unique identifier for this particular sequence instance.</p>
</td></tr>
  <tr><td>Note</td><td>This is a business identifer, not a resource identifier (see <a href="resource.html#identifiers">discussion</a>)</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#Identifier">Identifier</a></td></tr>
  <tr><td>Requirements</td><td><p>Allows sequences to be distinguished and referenced.</p>
</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.type"> </a><b>Sequence.type</b></td></tr>
  <tr><td>Definition</td><td><p>Amino acid / cDNA transcript / RNA variant.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>1..1</td></tr>
  <tr><td><a href="terminologies.html">Binding</a></td><td>sequenceType: <a href="valueset-sequence-type.html">Type if a sequence -- DNA, RNA, or amino acid sequence</a> (<a href="terminologies.html#example">Example</a>)</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#code">code</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.coordinateSystem"> </a><b>Sequence.coordinateSystem</b></td></tr>
  <tr><td>Definition</td><td><p>Whether the sequence is numbered starting at 0 (0-based numbering or coordinates) or starting at 1 (1-based numbering). Values are &quot;0&quot; for 0-based numbering and &quot;1&quot; for one-based.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>1..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.patient"> </a><b>Sequence.patient</b></td></tr>
  <tr><td>Definition</td><td><p>The patient whose sequencing results are described by this resource.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="references.html#Reference">Reference</a>(<a href="patient.html">Patient</a>)</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.specimen"> </a><b>Sequence.specimen</b></td></tr>
  <tr><td>Definition</td><td><p>Specimen used for sequencing.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="references.html#Reference">Reference</a>(<a href="specimen.html">Specimen</a>)</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.device"> </a><b>Sequence.device</b></td></tr>
  <tr><td>Definition</td><td><p>The method for sequencing, for example, chip information.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="references.html#Reference">Reference</a>(<a href="device.html">Device</a>)</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quantity"> </a><b>Sequence.quantity</b></td></tr>
  <tr><td>Definition</td><td><p>Quantity of the sequence.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#Quantity">Quantity</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq"> </a><b>Sequence.referenceSeq</b></td></tr>
  <tr><td>Definition</td><td><p>A reference sequence is a sequence that is used to represent an allele or variant.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq.chromosome"> </a><b>Sequence.referenceSeq.chromosome</b></td></tr>
  <tr><td>Definition</td><td><p>Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication (<a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0000340">SO:0000340 <img src="external.png" style="text-align: baseline"/></a>).</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="terminologies.html">Binding</a></td><td>chromosome-human: <a href="valueset-chromosome-human.html">Chromosome number for human</a> (<a href="terminologies.html#example">Example</a>)</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq.genomeBuild"> </a><b>Sequence.referenceSeq.genomeBuild</b></td></tr>
  <tr><td>Definition</td><td><p>The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq.referenceSeqId"> </a><b>Sequence.referenceSeq.referenceSeqId</b></td></tr>
  <tr><td>Definition</td><td><p>Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG<em>” identifies reference sequence for genes, “NM</em>” for messenger RNA transcripts, and “NP_” for amino acid sequences.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>1..1</td></tr>
  <tr><td><a href="terminologies.html">Binding</a></td><td>ENSEMBL: <a href="valueset-sequence-referenceSeq.html">Reference identifier</a> (<a href="terminologies.html#example">Example</a>)</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq.referenceSeqPointer"> </a><b>Sequence.referenceSeq.referenceSeqPointer</b></td></tr>
  <tr><td>Definition</td><td><p>A Pointer to another Sequence entity as refence sequence.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="references.html#Reference">Reference</a>(<a href="sequence.html">Sequence</a>)</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq.referenceSeqString"> </a><b>Sequence.referenceSeq.referenceSeqString</b></td></tr>
  <tr><td>Definition</td><td><p>A Reference Sequence string.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq.strand"> </a><b>Sequence.referenceSeq.strand</b></td></tr>
  <tr><td>Definition</td><td><p>Strand of DNA. Available values are &quot;1&quot; for the plus strand and &quot;-1&quot; for the minus strand.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>1..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq.windowStart"> </a><b>Sequence.referenceSeq.windowStart</b></td></tr>
  <tr><td>Definition</td><td><p>Start position (inclusive) of the window on the reference sequence.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>1..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq.windowEnd"> </a><b>Sequence.referenceSeq.windowEnd</b></td></tr>
  <tr><td>Definition</td><td><p>End position (exclusive) of the window on the reference sequence.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>1..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.variant"> </a><b>Sequence.variant</b></td></tr>
  <tr><td>Definition</td><td><p>A' is a variant (mutation) of A = definition every instance of A' is either an immediate mutation of some instance of A, or there is a chain of immediate mutation processes linking A' to some instance of A (<a href="http://www.sequenceontology.org/browser/current_svn/term/variant_of">variant_of <img src="external.png" style="text-align: baseline"/></a>).</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.variant.start"> </a><b>Sequence.variant.start</b></td></tr>
  <tr><td>Definition</td><td><p>Start position (inclusive) of the variant on the  reference sequence.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.variant.end"> </a><b>Sequence.variant.end</b></td></tr>
  <tr><td>Definition</td><td><p>End position (exclusive) of the variant on the reference sequence.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.variant.observedAllele"> </a><b>Sequence.variant.observedAllele</b></td></tr>
  <tr><td>Definition</td><td><p>An allele is one of a set of coexisting sequence variants of a gene (<a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0001023">SO:0001023 <img src="external.png" style="text-align: baseline"/></a>). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.variant.referenceAllele"> </a><b>Sequence.variant.referenceAllele</b></td></tr>
  <tr><td>Definition</td><td><p>An allele is one of a set of coexisting sequence variants of a gene (<a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0001023">SO:0001023 <img src="external.png" style="text-align: baseline"/></a>). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.variant.cigar"> </a><b>Sequence.variant.cigar</b></td></tr>
  <tr><td>Definition</td><td><p>Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation <a href="http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm">here <img src="external.png" style="text-align: baseline"/></a>.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.variant.variantPointer"> </a><b>Sequence.variant.variantPointer</b></td></tr>
  <tr><td>Definition</td><td><p>A pointer to an Observation containing variant information.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="references.html#Reference">Reference</a>(<a href="observation.html">Observation</a>)</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.observedSeq"> </a><b>Sequence.observedSeq</b></td></tr>
  <tr><td>Definition</td><td><p>Sequence that was observed.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality"> </a><b>Sequence.quality</b></td></tr>
  <tr><td>Definition</td><td><p>An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score (<a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0001686">SO:0001686 <img src="external.png" style="text-align: baseline"/></a>).</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.standardSequence"> </a><b>Sequence.quality.standardSequence</b></td></tr>
  <tr><td>Definition</td><td><p>Gold standard sequence used for comparing against.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.start"> </a><b>Sequence.quality.start</b></td></tr>
  <tr><td>Definition</td><td><p>Start position (inclusive) of the sequence.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.end"> </a><b>Sequence.quality.end</b></td></tr>
  <tr><td>Definition</td><td><p>End position (exclusive) of the sequence.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.score"> </a><b>Sequence.quality.score</b></td></tr>
  <tr><td>Definition</td><td><p>The score of an experimentally derived feature such as a p-value (<a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0001685">SO:0001685 <img src="external.png" style="text-align: baseline"/></a>).</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#Quantity">Quantity</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.method"> </a><b>Sequence.quality.method</b></td></tr>
  <tr><td>Definition</td><td><p>Method for quality.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.truthTP"> </a><b>Sequence.quality.truthTP</b></td></tr>
  <tr><td>Definition</td><td><p>True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.queryTP"> </a><b>Sequence.quality.queryTP</b></td></tr>
  <tr><td>Definition</td><td><p>True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.truthFN"> </a><b>Sequence.quality.truthFN</b></td></tr>
  <tr><td>Definition</td><td><p>False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.queryFP"> </a><b>Sequence.quality.queryFP</b></td></tr>
  <tr><td>Definition</td><td><p>False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.gtFP"> </a><b>Sequence.quality.gtFP</b></td></tr>
  <tr><td>Definition</td><td><p>The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar).</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.precision"> </a><b>Sequence.quality.precision</b></td></tr>
  <tr><td>Definition</td><td><p>QUERY.TP / (QUERY.TP + QUERY.FP).</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.recall"> </a><b>Sequence.quality.recall</b></td></tr>
  <tr><td>Definition</td><td><p>TRUTH.TP / (TRUTH.TP + TRUTH.FN).</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.fScore"> </a><b>Sequence.quality.fScore</b></td></tr>
  <tr><td>Definition</td><td><p>Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall).</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.readCoverage"> </a><b>Sequence.readCoverage</b></td></tr>
  <tr><td>Definition</td><td><p>Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.repository"> </a><b>Sequence.repository</b></td></tr>
  <tr><td>Definition</td><td><p>Configurations of the external repository.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.repository.url"> </a><b>Sequence.repository.url</b></td></tr>
  <tr><td>Definition</td><td><p>URI of an external repository which contains further details about the genetics data.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#uri">uri</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.repository.name"> </a><b>Sequence.repository.name</b></td></tr>
  <tr><td>Definition</td><td><p>URI of an external repository which contains further details about the genetics data.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.repository.variantId"> </a><b>Sequence.repository.variantId</b></td></tr>
  <tr><td>Definition</td><td><p>Id of the variant in this external repository.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.repository.readId"> </a><b>Sequence.repository.readId</b></td></tr>
  <tr><td>Definition</td><td><p>Id of the read in this external repository.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.pointer"> </a><b>Sequence.pointer</b></td></tr>
  <tr><td>Definition</td><td><p>Pointer to next atomic sequence which at most contains one variant.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="references.html#Reference">Reference</a>(<a href="sequence.html">Sequence</a>)</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant"> </a><b>Sequence.structureVariant</b></td></tr>
  <tr><td>Definition</td><td><p>Structural variant.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant.precisionOfBoundaries"> </a><b>Sequence.structureVariant.precisionOfBoundaries</b></td></tr>
  <tr><td>Definition</td><td><p>Precision of boundaries.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant.reportedaCGHRatio"> </a><b>Sequence.structureVariant.reportedaCGHRatio</b></td></tr>
  <tr><td>Definition</td><td><p>Structural Variant reported aCGH ratio.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant.length"> </a><b>Sequence.structureVariant.length</b></td></tr>
  <tr><td>Definition</td><td><p>Structural Variant Length.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant.outer"> </a><b>Sequence.structureVariant.outer</b></td></tr>
  <tr><td>Definition</td><td><p>Structural variant outer.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant.outer.start"> </a><b>Sequence.structureVariant.outer.start</b></td></tr>
  <tr><td>Definition</td><td><p>Structural Variant Outer Start-End.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant.outer.end"> </a><b>Sequence.structureVariant.outer.end</b></td></tr>
  <tr><td>Definition</td><td><p>Structural Variant Outer Start-End.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant.inner"> </a><b>Sequence.structureVariant.inner</b></td></tr>
  <tr><td>Definition</td><td><p>Structural variant inner.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant.inner.start"> </a><b>Sequence.structureVariant.inner.start</b></td></tr>
  <tr><td>Definition</td><td><p>Structural Variant Inner Start-End.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant.inner.end"> </a><b>Sequence.structureVariant.inner.end</b></td></tr>
  <tr><td>Definition</td><td><p>Structural Variant Inner Start-End.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
</table>



</div>


				</div>  <!-- /inner-wrapper -->
            </div>  <!-- /row -->
        </div>  <!-- /container -->
        
    </div>  <!-- /segment-content -->


	<div id="segment-footer" class="segment">  <!-- segment-footer -->
		<div class="container">  <!-- container -->
			<div class="inner-wrapper">
				<p>
        &copy; HL7.org 2011+. FHIR STU3 Candidate (v1.7.0-10082) generated on Thu, Oct 27, 2016 16:28+0000. <a href="qa.html">QA Page</a>
        <br/>
        <span style="color: #FFFF77">
        Links: <a style="color: #81BEF7" href="http://hl7.org/fhir/search.cfm">Search <img src="external.png" style="text-align: baseline"/></a> | 
               <a style="color: #81BEF7" href="history.html">Version History</a> | 
               <a style="color: #81BEF7" href="toc.html">Table of Contents</a> | 
               <a style="color: #81BEF7" href="http://services.w3.org/htmldiff?doc1=http%3A%2F%2Fhl7.org%2Ffhir%2FDSTU2%2Fsequence-definitions.html&amp;doc2=http%3A%2F%2Fhl7-fhir.github.io%2F%2Fsequence-definitions.html">Compare to DSTU2 <img src="external.png" style="text-align: baseline"/></a> |                
               <a rel="license" style="color: #81BEF7" href="license.html"><img src="cc0.png" alt="CC0" style="border-style: none;"/></a> | 
               <a style="color: #81BEF7" href="http://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemAdd&amp;tracker_id=677" target="_blank">Propose a change <img src="external.png" style="text-align: baseline"/></a>   
        </span>
        </p>
			</div>  <!-- /inner-wrapper -->
		</div>  <!-- /container -->
	</div>  <!-- /segment-footer -->
  <!-- disqus thread -->
  <!-- disqus -->
  <!-- end disqus -->        
	
	<div id="segment-post-footer" class="segment hidden">  <!-- segment-post-footer -->
		<div class="container">  <!-- container -->
		</div>  <!-- /container -->
	</div>  <!-- /segment-post-footer -->
    
      <!-- JS and analytics only. -->
      <!-- Bootstrap core JavaScript
================================================== -->
  <!-- Placed at the end of the document so the pages load faster -->
<script src="./assets/js/jquery.js"> </script>     <!-- note keep space here, otherwise it will be transformed to empty tag -> fails -->
<script src="./dist/js/bootstrap.min.js"> </script>
<script src="./assets/js/respond.min.js"> </script>

<script src="./assets/js/fhir.js"> </script>

  <!-- Analytics Below
================================================== -->

    
</body>
</html>