<!DOCTYPE HTML>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>Sequence - FHIR v1.7.0</title>

  
  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
  <meta name="author" content="http://hl7.org/fhir"/>

  <link rel="stylesheet" href="fhir.css"/>
  <link rel="Prev" href="http://hl7.org/fhir/sequence.html"/>

    <!-- Bootstrap core CSS -->
  <link rel="stylesheet" href="./dist/css/bootstrap.css"/>
  <link rel="stylesheet" href="./assets/css/bootstrap-fhir.css"/>

    <!-- Project extras -->
  <link rel="stylesheet" href="./assets/css/project.css"/>
  <link rel="stylesheet" href="./assets/css/pygments-manni.css"/>
	<link rel="stylesheet" href="jquery-ui.css"/>

    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- [if lt IE 9]>
  <script src="./assets/js/html5shiv.js"></script>
  <script src="./assets/js/respond.min.js"></script>
  <![endif] -->

    <!-- Favicons -->
  <link sizes="144x144" rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-144-precomposed.png"/>
  <link sizes="114x114" rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-114-precomposed.png"/>
  <link sizes="72x72" rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-72-precomposed.png"/>
  <link rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-57-precomposed.png"/>
  <link rel="shortcut icon" href="./assets/ico/favicon.png"/>

</head>
<body>
	<div id="segment-header" class="segment">  <!-- segment-header -->	
		<div class="container">  <!-- container -->
			<a no-external="true" id="logo" href="http://hl7.org/fhir"><img src="./assets/images/fhir-logo-www.png" alt="logo fhir"/> </a>
			<div id="hl7-status">
				<b>Current Build</b>
			</div>       	
			
			<div id="hl7-nav">
				 <a no-external="true" id="hl7-logo" href="http://www.hl7.org">
					<img src="./assets/images/hl7-logo.png" width="42" alt="visit the hl7 website" height="50"/>
				</a> 
			</div>       	
			<div id="hl7-nav"><a no-external="true" id="hl7-logo" href="http://hl7.org/fhir/search.cfm"><img src="./assets/images/search.png" alt="Search FHIR"/></a></div> 
		</div>
		<div class="container">  <!-- container -->
	</div></div>  <!-- /segment-header -->	


	<div id="segment-navbar" class="segment">  <!-- segment-navbar -->
		<div id="stripe"> </div>
		<div class="container">  <!-- container -->
   <!-- HEADER CONTENT -->
 	      		 	      		      		
			<nav class="navbar navbar-inverse">
				<div class="container">
					<button data-target=".navbar-inverse-collapse" data-toggle="collapse" type="button" class="navbar-toggle">
						<span class="icon-bar"> </span>
						<span class="icon-bar"> </span>
						<span class="icon-bar"> </span>
					</button>
					<a href="index.html" class="navbar-brand hidden">FHIR</a>
					<div class="nav-collapse collapse navbar-inverse-collapse">
						<ul class="nav navbar-nav">
              <li><a href="./index.html">Home</a></li>
              <li><a href="./modules.html">Getting Started</a></li>
              <li><a href="./documentation.html">Documentation</a></li>
              <li><a href="./resourcelist.html">Resources</a></li>
              <li><a href="./profilelist.html">Profiles</a></li>
              <li><a href="./extensibility-registry.html">Extensions</a></li>
              <li><a href="./operationslist.html">Operations</a></li>
              <li><a href="./servicelist.html">Services</a></li>
						</ul>
					</div>  <!-- /.nav-collapse -->
				</div>  <!-- /.container -->
			</nav>  <!-- /.navbar -->
      		
  <!-- /HEADER CONTENT -->				
		</div>  <!-- /container -->
	</div>  <!-- /segment-navbar -->
	

	<div id="segment-breadcrumb" class="segment">  <!-- segment-breadcrumb -->
		<div class="container">  <!-- container -->	
			<ul class="breadcrumb">
        <li><a href="diagnostics-module.html"><img src="diagnostics.jpg"/> Diagnostics</a></li>        <li><b>Sequence</b></li>          <!-- sequence.html / resource / Sequence -->

			</ul>	
		</div>  <!-- /container -->
	</div>  <!-- /segment-breadcrumb -->
	

	<div id="segment-content" class="segment">  <!-- segment-content -->
	<div class="container">  <!-- container -->
            <div class="row">
            	<div class="inner-wrapper">
  <!-- CONTENT CONTENT -->


<div class="col-9">

<ul class="nav nav-tabs"><li class="active"><a href="#">Content</a></li><li><a href="sequence-examples.html">Examples</a></li><li><a href="sequence-definitions.html">Detailed Descriptions</a></li><li><a href="sequence-mappings.html">Mappings</a></li><li><a href="sequence-profiles.html">Profiles &amp; Extensions</a></li></ul>

<h1 class="self-link-parent"><span class="sectioncount">10.4<a name="10.4"> </a></span> Resource Sequence - Content <a href="sequence.html#10.4" title="link to here" class="self-link"><img src="assets/images/link.svg" width="20" class="self-link" height="20"/></a></h1>


<table class="cols">
 <tr><td><a href="http://www.hl7.org/Special/committees/clingenomics/index.cfm" _target="blank">Clinical Genomics <img src="external.png" style="text-align: baseline"/></a> Work Group</td><td><a href="resource.html#maturity">Maturity Level</a>: 0</td><td><a href="compartmentdefinition.html">Compartments</a>: Not linked to any defined compartments</td></tr>
</table>



<p> 
Raw data describing a biological sequence. 
</p>

<div>
    
<h2 class="self-link-parent"><span class="sectioncount">10.4.1<a name="10.4.1"> </a></span> Scope and Usage <a href="sequence.html#10.4.1" title="link to here" class="self-link"><img src="assets/images/link.svg" width="20" class="self-link" height="20"/></a></h2>
<p>
    The Sequence resource is designed to describe an atomic sequence which contains more than one bases but has at most one variation. Atomic sequences can be connected by link element and they will lead to sequence graph. By this method, a sequence can be reported. Complete genetic sequence information, of which specific genetic variations are a part, is reported by reference to the GA4GH repository.  Thus, the FHIR Sequence resource avoids large genomic payloads in a manner analogous to how the FHIR ImagingStudy resource references large images maintained in other systems.
    This resource contextualizes well established standards from the field of clinical genetics into the standards of healthcare (e.g. HGNC - HUGO Gene Nomenclature Committee's international standard for gene names, symbols, and identifiers).
</p>
<h3 class="self-link-parent"><span class="sectioncount">10.4.1.1<a name="10.4.1.1"> </a></span> Genetic Standards and Resources include: <a href="sequence.html#10.4.1.1" title="link to here" class="self-link"><img src="assets/images/link.svg" width="20" class="self-link" height="20"/></a></h3>
<ul>
    <li>Variant Databases: <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi">dbSNP <img src="external.png" style="text-align: baseline"/></a> , <a href="http://www.ncbi.nlm.nih.gov/clinvar/">ClinVar <img src="external.png" style="text-align: baseline"/></a>, and <a href="http://cancer.sanger.ac.uk/cosmic/">COSMIC <img src="external.png" style="text-align: baseline"/></a> </li>
    <li>Reference Sequences: <a href="http://www.ncbi.nlm.nih.gov/refseq/">RefSeq <img src="external.png" style="text-align: baseline"/></a> and <a href="http://www.ensembl.org/index.html">ENSEMBL <img src="external.png" style="text-align: baseline"/></a> </li>
</ul>
<p>
    This resource is designed to describe a sequence variation with clinical significance with information such as:
</p>
    <ul>
        <li>Name of the variation represented</li>
        <li>Type of the variation</li>
        <li>Gene region occupied by the variation</li>
        <li>Name of the gene</li>
        <li>Tissue source used to determine genotype of the variation</li>
    </ul>
<p>
    It is strongly encouraged to provide all available information in this resource for any reported variants, because receiving systems (e.g. discovery research, outcomes analysis, and public health reporting) may use this information to normalize variants over time or across sources.  However, these data should not be used to dynamically correct/change variant representations for clinical use outside of the laboratory, due to insufficient information.
</p>
<p>
    Implementers should be aware that semantic equivalency of results of genetic variants cannot be guaranteed unless there is an agreed upon standard between sending and receiving systems.
</p>

</div>

<div>
<h2 class="self-link-parent"><span class="sectioncount">10.4.2<a name="10.4.2"> </a></span> Boundaries and Relationships <a href="sequence.html#10.4.2" title="link to here" class="self-link"><img src="assets/images/link.svg" width="20" class="self-link" height="20"/></a></h2>

<p>
    Focus of the resource is to provide data <b>immediately</b> relevant to clinical decision-making. Hence data such as precise read of DNA sequences and sequence alignment are not included; such data are nonetheless accessible through references to <a href="http://ga4gh.org/#/">GA4GH <img src="external.png" style="text-align: baseline"/></a> (Global Alliance for Genomics and Health) API.
    
    The Sequence resource will be referenced by <a href="observation.html">Observation</a> to provide variant information. As clinical assessments/diagnosis of a patient are typically captured in the <a href="condition.html">Condition</a> resource or the ClinicalImpression resource, the Sequence resource can be referenced by the Condition resource to provide specific genetic data to support an assertions. This is analogous to how Condition references other resources, such as <a href="allergyintolerance.html">AllergyIntolerance</a>, <a href="procedure.html">Procedure</a>, and <a href="questionnaire.html">Questionnaire</a> resources.
</p>

</div>

  <!-- (optional):
<div>
<h2>Background and Context</h2>
<p>
Provides additional detail on exactly how the resource is to be used
</p>
</div> -->



<p>This resource is referenced by <a href="observation.html">observation</a></p>


<a name="resource"/>
<h2 class="self-link-parent"><span class="sectioncount">10.4.3</span> 
Resource Content
 <a href="sequence.html#resource" title="link to here" class="self-link"><img src="assets/images/link.svg" width="20" class="self-link" height="20"/></a></h2>
<a name="def"> </a>
<a name="sequence"> </a>
<a name="Sequence"> </a>

<div id="tabs">
  <ul>
    <li><a href="#tabs-struc">Structure</a></li>
    <li><a href="#tabs-uml">UML</a></li>
    <li><a href="#tabs-xml">XML</a></li>
    <li><a href="#tabs-json">JSON</a></li>
    <li><a href="#tabs-ttl">Turtle</a></li>
    <li><a href="#tabs-diff">R2 Diff</a></li>
    <li><a href="#tabs-all">All</a></li>
  </ul>
  <div id="tabs-struc">

<div id="tbl">
<p><b>Structure</b></p>
 <div id="tbl-inner">
  <table border="0" cellpadding="0" cellspacing="0" style="border: 0px; font-size: 11px; font-family: verdana; vertical-align: top;"><tr style="border: 1px #F0F0F0 solid; font-size: 11px; font-family: verdana; vertical-align: top;"><th style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="formats.html#table" title="The logical name of the element">Name</a></th><th style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="formats.html#table" title="Information about the use of the element">Flags</a></th><th style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="formats.html#table" title="Minimum and Maximum # of times the the element can appear in the instance">Card.</a></th><th style="width: 100px" class="hierarchy"><a href="formats.html#table" title="Reference to the type of the element">Type</a></th><th style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="formats.html#table" title="Additional information about the element">Description &amp; Constraints</a><span style="float: right"><a href="formats.html#table" title="Legend for this format"><img src="help16.png" alt="doco" style="background-color: inherit"/></a></span></th></tr><tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck1.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_resource.png" alt="." style="background-color: white; background-color: inherit" title="Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence" title="Sequence : Raw data describing a biological sequence.">Sequence</a><a name="Sequence"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"/><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="domainresource.html">DomainResource</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Information about a biological sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.identifier" title="Sequence.identifier : A unique identifier for this particular sequence instance.">identifier</a><a name="Sequence.identifier"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#Identifier">Identifier</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Unique ID for this particular sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.type" title="Sequence.type : Amino acid / cDNA transcript / RNA variant.">type</a><a name="Sequence.type"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#code">code</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">AA | DNA | RNA<br/><a href="valueset-sequence-type.html" title="Type if a sequence -- DNA, RNA, or amino acid sequence">sequenceType</a> (<a href="terminologies.html#example" title="Instances are not expected or even encouraged to draw from the specified value set.  The value set merely provides examples of the types of concepts intended to be included.">Example</a>)</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.coordinateSystem" title="Sequence.coordinateSystem : Whether the sequence is numbered starting at 0 (0-based numbering or coordinates) or starting at 1 (1-based numbering). Values are &quot;0&quot; for 0-based numbering and &quot;1&quot; for one-based.">coordinateSystem</a><a name="Sequence.coordinateSystem"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Numbering used for sequence (0-based or 1-based)</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.patient" title="Sequence.patient : The patient whose sequencing results are described by this resource.">patient</a><a name="Sequence.patient"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html">Reference</a>(<a href="patient.html">Patient</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Who and/or what this is about</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.specimen" title="Sequence.specimen : Specimen used for sequencing.">specimen</a><a name="Sequence.specimen"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html">Reference</a>(<a href="specimen.html">Specimen</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Specimen used for sequencing</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.device" title="Sequence.device : The method for sequencing, for example, chip information.">device</a><a name="Sequence.device"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html">Reference</a>(<a href="device.html">Device</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">The method for sequencing</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quantity" title="Sequence.quantity : Quantity of the sequence.">quantity</a><a name="Sequence.quantity"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#Quantity">Quantity</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Quantity of the sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck11.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq" title="Sequence.referenceSeq : A reference sequence is a sequence that is used to represent an allele or variant.">referenceSeq</a><a name="Sequence.referenceSeq"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Reference sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.chromosome" title="Sequence.referenceSeq.chromosome : Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340)).">chromosome</a><a name="Sequence.referenceSeq.chromosome"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Chromosome containing genetic finding<br/><a href="valueset-chromosome-human.html" title="Chromosome number for human">chromosome-human</a> (<a href="terminologies.html#example" title="Instances are not expected or even encouraged to draw from the specified value set.  The value set merely provides examples of the types of concepts intended to be included.">Example</a>)</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild" title="Sequence.referenceSeq.genomeBuild : The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used.">genomeBuild</a><a name="Sequence.referenceSeq.genomeBuild"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId" title="Sequence.referenceSeq.referenceSeqId : Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences.">referenceSeqId</a><a name="Sequence.referenceSeq.referenceSeqId"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Reference identifier<br/><a href="valueset-sequence-referenceSeq.html" title="Reference identifier">ENSEMBL</a> (<a href="terminologies.html#example" title="Instances are not expected or even encouraged to draw from the specified value set.  The value set merely provides examples of the types of concepts intended to be included.">Example</a>)</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer" title="Sequence.referenceSeq.referenceSeqPointer : A Pointer to another Sequence entity as refence sequence.">referenceSeqPointer</a><a name="Sequence.referenceSeq.referenceSeqPointer"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html">Reference</a>(<a href="sequence.html">Sequence</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">A Pointer to another Sequence entity as refence sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString" title="Sequence.referenceSeq.referenceSeqString : A Reference Sequence string.">referenceSeqString</a><a name="Sequence.referenceSeq.referenceSeqString"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">A Reference Sequence string</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.strand" title="Sequence.referenceSeq.strand : Strand of DNA. Available values are &quot;1&quot; for the plus strand and &quot;-1&quot; for the minus strand.">strand</a><a name="Sequence.referenceSeq.strand"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Strand of DNA</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.windowStart" title="Sequence.referenceSeq.windowStart : Start position (inclusive) of the window on the reference sequence.">windowStart</a><a name="Sequence.referenceSeq.windowStart"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Start position (inclusive) of the window on the  reference sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck100.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.windowEnd" title="Sequence.referenceSeq.windowEnd : End position (exclusive) of the window on the reference sequence.">windowEnd</a><a name="Sequence.referenceSeq.windowEnd"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">End position (exclusive) of the window on the reference sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck11.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant" title="Sequence.variant : A' is a variant (mutation) of A = definition every instance of A' is either an immediate mutation of some instance of A, or there is a chain of immediate mutation processes linking A' to some instance of A ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of)).">variant</a><a name="Sequence.variant"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Sequence variant</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.start" title="Sequence.variant.start : Start position (inclusive) of the variant on the  reference sequence.">start</a><a name="Sequence.variant.start"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Start position (inclusive) of the variant on the  reference sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.end" title="Sequence.variant.end : End position (exclusive) of the variant on the reference sequence.">end</a><a name="Sequence.variant.end"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">End position (exclusive) of the variant on the reference sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.observedAllele" title="Sequence.variant.observedAllele : An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand.">observedAllele</a><a name="Sequence.variant.observedAllele"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Allele that was observed</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.referenceAllele" title="Sequence.variant.referenceAllele : An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand.">referenceAllele</a><a name="Sequence.variant.referenceAllele"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Allele of reference sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.cigar" title="Sequence.variant.cigar : Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm).">cigar</a><a name="Sequence.variant.cigar"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Extended CIGAR string for aligning the sequence with reference bases</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck100.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.variantPointer" title="Sequence.variant.variantPointer : A pointer to an Observation containing variant information.">variantPointer</a><a name="Sequence.variant.variantPointer"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html">Reference</a>(<a href="observation.html">Observation</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Pointer to observed variant information</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.observedSeq" title="Sequence.observedSeq : Sequence that was observed.">observedSeq</a><a name="Sequence.observedSeq"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Observed sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck11.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality" title="Sequence.quality : An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686)).">quality</a><a name="Sequence.quality"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Sequence quality</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.standardSequence" title="Sequence.quality.standardSequence : Gold standard sequence used for comparing against.">standardSequence</a><a name="Sequence.quality.standardSequence"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Standard sequence for comparison</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.start" title="Sequence.quality.start : Start position (inclusive) of the sequence.">start</a><a name="Sequence.quality.start"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Start position (inclusive) of the sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.end" title="Sequence.quality.end : End position (exclusive) of the sequence.">end</a><a name="Sequence.quality.end"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">End position (exclusive) of the sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.score" title="Sequence.quality.score : The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685)).">score</a><a name="Sequence.quality.score"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#Quantity">Quantity</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Quality score</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.method" title="Sequence.quality.method : Method for quality.">method</a><a name="Sequence.quality.method"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Method for quality</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.truthTP" title="Sequence.quality.truthTP : True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event.">truthTP</a><a name="Sequence.quality.truthTP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">True positives from the perspective of the truth data</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.queryTP" title="Sequence.quality.queryTP : True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event.">queryTP</a><a name="Sequence.quality.queryTP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">True positives from the perspective of the query data</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.truthFN" title="Sequence.quality.truthFN : False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here.">truthFN</a><a name="Sequence.quality.truthFN"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">False negatives</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.queryFP" title="Sequence.quality.queryFP : False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here.">queryFP</a><a name="Sequence.quality.queryFP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">False positives</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.gtFP" title="Sequence.quality.gtFP : The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar).">gtFP</a><a name="Sequence.quality.gtFP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">False positives where the non-REF alleles in the Truth and Query Call Sets match</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.precision" title="Sequence.quality.precision : QUERY.TP / (QUERY.TP + QUERY.FP).">precision</a><a name="Sequence.quality.precision"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Precision</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.recall" title="Sequence.quality.recall : TRUTH.TP / (TRUTH.TP + TRUTH.FN).">recall</a><a name="Sequence.quality.recall"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Recall</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck100.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.fScore" title="Sequence.quality.fScore : Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall).">fScore</a><a name="Sequence.quality.fScore"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">F-score</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.readCoverage" title="Sequence.readCoverage : Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence.">readCoverage</a><a name="Sequence.readCoverage"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Average number of reads representing a given nucleotide in the reconstructed sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck11.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository" title="Sequence.repository : Configurations of the external repository.">repository</a><a name="Sequence.repository"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">External repository</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.url" title="Sequence.repository.url : URI of an external repository which contains further details about the genetics data.">url</a><a name="Sequence.repository.url"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#uri">uri</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">URI of the repository</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.name" title="Sequence.repository.name : URI of an external repository which contains further details about the genetics data.">name</a><a name="Sequence.repository.name"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Name of the repository</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.variantId" title="Sequence.repository.variantId : Id of the variant in this external repository.">variantId</a><a name="Sequence.repository.variantId"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Id of the variant</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck100.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.readId" title="Sequence.repository.readId : Id of the read in this external repository.">readId</a><a name="Sequence.repository.readId"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Id of the read</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.pointer" title="Sequence.pointer : Pointer to next atomic sequence which at most contains one variant.">pointer</a><a name="Sequence.pointer"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html">Reference</a>(<a href="sequence.html">Sequence</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Pointer to next atomic sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck01.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant" title="Sequence.structureVariant : Structural variant.">structureVariant</a><a name="Sequence.structureVariant"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Structural variant</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.precisionOfBoundaries" title="Sequence.structureVariant.precisionOfBoundaries : Precision of boundaries.">precisionOfBoundaries</a><a name="Sequence.structureVariant.precisionOfBoundaries"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Precision of boundaries</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio" title="Sequence.structureVariant.reportedaCGHRatio : Structural Variant reported aCGH ratio.">reportedaCGHRatio</a><a name="Sequence.structureVariant.reportedaCGHRatio"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant reported aCGH ratio</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.length" title="Sequence.structureVariant.length : Structural Variant Length.">length</a><a name="Sequence.structureVariant.length"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Length</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck011.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.outer" title="Sequence.structureVariant.outer : Structural variant outer.">outer</a><a name="Sequence.structureVariant.outer"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Structural variant outer</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck0110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.outer.start" title="Sequence.structureVariant.outer.start : Structural Variant Outer Start-End.">start</a><a name="Sequence.structureVariant.outer.start"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Outer Start-End</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck0100.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.outer.end" title="Sequence.structureVariant.outer.end : Structural Variant Outer Start-End.">end</a><a name="Sequence.structureVariant.outer.end"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Outer Start-End</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck001.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.inner" title="Sequence.structureVariant.inner : Structural variant inner.">inner</a><a name="Sequence.structureVariant.inner"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Structural variant inner</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck0010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.inner.start" title="Sequence.structureVariant.inner.start : Structural Variant Inner Start-End.">start</a><a name="Sequence.structureVariant.inner.start"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Inner Start-End</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck0000.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.inner.end" title="Sequence.structureVariant.inner.end : Structural Variant Inner Start-End.">end</a><a name="Sequence.structureVariant.inner.end"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Inner Start-End</td></tr>
<tr><td colspan="5" class="hierarchy"><br/><a href="formats.html#table" title="Legend for this format"><img src="help16.png" alt="doco" style="background-color: inherit"/> Documentation for this format</a></td></tr></table>
 </div>
</div>

  </div>
  <div id="tabs-uml">
  
<div id="uml">
<p><b>UML Diagram</b> (<a href="formats.html#uml">Legend</a>)</p>
 <div id="uml-inner">
 <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="986.6" version="1.1" height="585.0">  <defs>
    <filter x="0" width="200%" y="0" id="shadowA" height="200%">
      <feOffset result="offOut" dx="3" in="SourceGraphic" dy="3"/>
      <feColorMatrix result="matrixOut" in="offOut" values="0.2 0 0 0 0 0 0.2 0 0 0 0 0 0.2 0 0 0 0 0 1 0" type="matrix"/>
      <feGaussianBlur result="blurOut" stdDeviation="2" in="matrixOut"/>
      <feBlend mode="normal" in="SourceGraphic" in2="blurOut"/>
    </filter>
  </defs><g><rect filter="url(#shadowA)" rx="4" ry="4" x="300.4" width="176.0" y="228.0" style="fill:#f0f8ff;stroke:black;stroke-width:1" height="176.0"/><line y1="256.0" x1="300.4" y2="256.0" x2="476.4" style="stroke:dimgrey;stroke-width:1"/><text x="388.4" y="248.0" fill="black" class="diagram-class-title  diagram-class-resource">Sequence<tspan class="diagram-class-title-link"> (<a class="diagram-class-reference" xlink:href="domainresource.html">DomainResource</a>)</tspan></text><text x="306.4" y="270.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.identifier"><title>A unique identifier for this particular sequence instance</title>identifier</a> : <a xlink:href="datatypes.html#Identifier">Identifier</a> [0..*]</text><text x="306.4" y="284.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.type"><title>Amino acid / cDNA transcript / RNA variant</title>type</a> : <a xlink:href="datatypes.html#code">code</a> [1..1] « <a xlink:href="valueset-sequence-type.html"><title>Type if a sequence -- DNA, RNA, or amino acid sequence (Strength=Example)</title>sequenceType</a>?? »</text><text x="306.4" y="298.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.coordinateSystem"><title>Whether the sequence is numbered starting at 0 (0-based numbering or coordinates) or starting at 1 (1-based numbering). Values are &quot;0&quot; for 0-based numbering and &quot;1&quot; for one-based</title>coordinateSystem</a> : <a xlink:href="datatypes.html#integer">integer</a> [1..1]</text><text x="306.4" y="312.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.patient"><title>The patient whose sequencing results are described by this resource</title>patient</a> : <a xlink:href="references.html#Reference">Reference</a> [0..1] « <a xlink:href="patient.html#Patient">Patient</a> »</text><text x="306.4" y="326.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.specimen"><title>Specimen used for sequencing</title>specimen</a> : <a xlink:href="references.html#Reference">Reference</a> [0..1] « <a xlink:href="specimen.html#Specimen">Specimen</a> »</text><text x="306.4" y="340.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.device"><title>The method for sequencing, for example, chip information</title>device</a> : <a xlink:href="references.html#Reference">Reference</a> [0..1] « <a xlink:href="device.html#Device">Device</a> »</text><text x="306.4" y="354.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quantity"><title>Quantity of the sequence</title>quantity</a> : <a xlink:href="datatypes.html#Quantity">Quantity</a> [0..1]</text><text x="306.4" y="368.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.observedSeq"><title>Sequence that was observed</title>observedSeq</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="306.4" y="382.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.readCoverage"><title>Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence</title>readCoverage</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="306.4" y="396.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.pointer"><title>Pointer to next atomic sequence which at most contains one variant</title>pointer</a> : <a xlink:href="references.html#Reference">Reference</a> [0..*] « <a xlink:href="sequence.html#Sequence">Sequence</a> »</text><g><rect filter="url(#shadowA)" rx="4" ry="4" x="265.19999999999993" width="246.40000000000003" y="0.0" style="fill:#f0f8ff;stroke:black;stroke-width:1" height="148.0"/><line y1="28.0" x1="265.19999999999993" y2="28.0" x2="511.59999999999997" style="stroke:dimgrey;stroke-width:1"/><text x="388.4" y="20.0" fill="black" class="diagram-class-title">ReferenceSeq</text><text x="271.19999999999993" y="42.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.referenceSeq.chromosome"><title>Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340))</title>chromosome</a> : <a xlink:href="datatypes.html#CodeableConcept">CodeableConcept</a> [0..1] « <a xlink:href="valueset-chromosome-human.html"><title>Chromosome number for human (Strength=Example)</title>chromosome-human</a>?? »</text><text x="271.19999999999993" y="56.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild"><title>The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used</title>genomeBuild</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="271.19999999999993" y="70.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId"><title>Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences</title>referenceSeqId</a> : <a xlink:href="datatypes.html#CodeableConcept">CodeableConcept</a> [1..1] « <a xlink:href="valueset-sequence-referenceSeq.html"><title>Reference identifier (Strength=Example)</title>ENSEMBL</a>?? »</text><text x="271.19999999999993" y="84.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer"><title>A Pointer to another Sequence entity as refence sequence</title>referenceSeqPointer</a> : <a xlink:href="references.html#Reference">Reference</a> [0..1] « <a xlink:href="sequence.html#Sequence">Sequence</a> »</text><text x="271.19999999999993" y="98.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString"><title>A Reference Sequence string</title>referenceSeqString</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="271.19999999999993" y="112.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.referenceSeq.strand"><title>Strand of DNA. Available values are &quot;1&quot; for the plus strand and &quot;-1&quot; for the minus strand</title>strand</a> : <a xlink:href="datatypes.html#integer">integer</a> [1..1]</text><text x="271.19999999999993" y="126.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.referenceSeq.windowStart"><title>Start position (inclusive) of the window on the reference sequence</title>windowStart</a> : <a xlink:href="datatypes.html#integer">integer</a> [1..1]</text><text x="271.19999999999993" y="140.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.referenceSeq.windowEnd"><title>End position (exclusive) of the window on the reference sequence</title>windowEnd</a> : <a xlink:href="datatypes.html#integer">integer</a> [1..1]</text></g><g><rect filter="url(#shadowA)" rx="4" ry="4" x="540.4" width="215.60000000000002" y="228.0" style="fill:#f0f8ff;stroke:black;stroke-width:1" height="120.0"/><line y1="256.0" x1="540.4" y2="256.0" x2="756.0" style="stroke:dimgrey;stroke-width:1"/><text x="648.2" y="248.0" fill="black" class="diagram-class-title">Variant</text><text x="546.4" y="270.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.variant.start"><title>Start position (inclusive) of the variant on the  reference sequence</title>start</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="546.4" y="284.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.variant.end"><title>End position (exclusive) of the variant on the reference sequence</title>end</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="546.4" y="298.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.variant.observedAllele"><title>An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand</title>observedAllele</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="546.4" y="312.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.variant.referenceAllele"><title>An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand</title>referenceAllele</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="546.4" y="326.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.variant.cigar"><title>Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm)</title>cigar</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="546.4" y="340.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.variant.variantPointer"><title>A pointer to an Observation containing variant information</title>variantPointer</a> : <a xlink:href="references.html#Reference">Reference</a> [0..1] « <a xlink:href="observation.html#Observation">Observation</a> »</text></g><g><rect filter="url(#shadowA)" rx="4" ry="4" x="0.0" width="180.4" y="207.0" style="fill:#f0f8ff;stroke:black;stroke-width:1" height="218.0"/><line y1="235.0" x1="0.0" y2="235.0" x2="180.4" style="stroke:dimgrey;stroke-width:1"/><text x="90.2" y="227.0" fill="black" class="diagram-class-title">Quality</text><text x="6.0" y="249.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.standardSequence"><title>Gold standard sequence used for comparing against</title>standardSequence</a> : <a xlink:href="datatypes.html#CodeableConcept">CodeableConcept</a> [0..1]</text><text x="6.0" y="263.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.start"><title>Start position (inclusive) of the sequence</title>start</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="6.0" y="277.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.end"><title>End position (exclusive) of the sequence</title>end</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="6.0" y="291.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.score"><title>The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685))</title>score</a> : <a xlink:href="datatypes.html#Quantity">Quantity</a> [0..1]</text><text x="6.0" y="305.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.method"><title>Method for quality</title>method</a> : <a xlink:href="datatypes.html#CodeableConcept">CodeableConcept</a> [0..1]</text><text x="6.0" y="319.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.truthTP"><title>True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event</title>truthTP</a> : <a xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="333.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.queryTP"><title>True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event</title>queryTP</a> : <a xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="347.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.truthFN"><title>False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here</title>truthFN</a> : <a xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="361.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.queryFP"><title>False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here</title>queryFP</a> : <a xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="375.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.gtFP"><title>The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar)</title>gtFP</a> : <a xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="389.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.precision"><title>QUERY.TP / (QUERY.TP + QUERY.FP)</title>precision</a> : <a xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="403.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.recall"><title>TRUTH.TP / (TRUTH.TP + TRUTH.FN)</title>recall</a> : <a xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="417.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.fScore"><title>Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall)</title>fScore</a> : <a xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text></g><g><rect filter="url(#shadowA)" rx="4" ry="4" x="596.4" width="110.00000000000001" y="350.0" style="fill:#f0f8ff;stroke:black;stroke-width:1" height="92.0"/><line y1="378.0" x1="596.4" y2="378.0" x2="706.4" style="stroke:dimgrey;stroke-width:1"/><text x="651.4" y="370.0" fill="black" class="diagram-class-title">Repository</text><text x="602.4" y="392.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.repository.url"><title>URI of an external repository which contains further details about the genetics data</title>url</a> : <a xlink:href="datatypes.html#uri">uri</a> [0..1]</text><text x="602.4" y="406.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.repository.name"><title>URI of an external repository which contains further details about the genetics data</title>name</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="602.4" y="420.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.repository.variantId"><title>Id of the variant in this external repository</title>variantId</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="602.4" y="434.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.repository.readId"><title>Id of the read in this external repository</title>readId</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text></g><g><rect filter="url(#shadowA)" rx="4" ry="4" x="307.0" width="162.8" y="484.0" style="fill:#f0f8ff;stroke:black;stroke-width:1" height="78.0"/><line y1="512.0" x1="307.0" y2="512.0" x2="469.8" style="stroke:dimgrey;stroke-width:1"/><text x="388.4" y="504.0" fill="black" class="diagram-class-title">StructureVariant</text><text x="313.0" y="526.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.structureVariant.precisionOfBoundaries"><title>Precision of boundaries</title>precisionOfBoundaries</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="313.0" y="540.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio"><title>Structural Variant reported aCGH ratio</title>reportedaCGHRatio</a> : <a xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="313.0" y="554.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.structureVariant.length"><title>Structural Variant Length</title>length</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text><g><rect filter="url(#shadowA)" rx="4" ry="4" x="90.19999999999996" width="96.80000000000001" y="491.0" style="fill:#f0f8ff;stroke:black;stroke-width:1" height="64.0"/><line y1="519.0" x1="90.19999999999996" y2="519.0" x2="186.99999999999997" style="stroke:dimgrey;stroke-width:1"/><text x="138.59999999999997" y="511.0" fill="black" class="diagram-class-title">Outer</text><text x="96.19999999999996" y="533.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.structureVariant.outer.start"><title>Structural Variant Outer Start-End</title>start</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="96.19999999999996" y="547.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.structureVariant.outer.end"><title>Structural Variant Outer Start-End</title>end</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text></g><g><rect filter="url(#shadowA)" rx="4" ry="4" x="589.8" width="96.80000000000001" y="491.0" style="fill:#f0f8ff;stroke:black;stroke-width:1" height="64.0"/><line y1="519.0" x1="589.8" y2="519.0" x2="686.5999999999999" style="stroke:dimgrey;stroke-width:1"/><text x="638.1999999999999" y="511.0" fill="black" class="diagram-class-title">Inner</text><text x="595.8" y="533.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.structureVariant.inner.start"><title>Structural Variant Inner Start-End</title>start</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="595.8" y="547.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.structureVariant.inner.end"><title>Structural Variant Inner Start-End</title>end</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text></g></g></g><line y1="228.0" x1="388.4" y2="147.99999999999997" x2="388.4" style="stroke:navy;stroke-width:1"/><polygon transform="rotate(-90.0 388.4 228.0)" style="fill:navy;stroke:navy;stroke-width:1" points="388.4,228.0 394.4,232.0 400.4,228.0 394.4,224.0 388.4,228.0"/><rect x="362.0" width="52.0" y="181.0" style="fill:white;stroke:black;stroke-width:0" height="18.0"/><text x="388.0" y="191.0" fill="black" class="diagram-class-linkage"><a xlink:href="sequence-definitions.html#Sequence.referenceSeq"><title>A reference sequence is a sequence that is used to represent an allele or variant</title>referenceSeq</a></text><text x="388.4" y="161.99999999999997" fill="black" class="diagram-class-linkage">[0..1]</text><line y1="306.5157813702848" x1="476.4" y2="299.6181678214011" x2="540.4" style="stroke:navy;stroke-width:1"/><polygon transform="rotate(-6.151321210260629 476.4 306.5157813702848)" style="fill:navy;stroke:navy;stroke-width:1" points="476.4,306.5157813702848 482.4,310.5157813702848 488.4,306.5157813702848 482.4,302.5157813702848 476.4,306.5157813702848"/><rect x="493.0" width="30.0" y="296.0" style="fill:white;stroke:black;stroke-width:0" height="18.0"/><text x="508.0" y="306.0" fill="black" class="diagram-class-linkage"><a xlink:href="sequence-definitions.html#Sequence.variant"><title>A' is a variant (mutation) of A = definition every instance of A' is either an immediate mutation of some instance of A, or there is a chain of immediate mutation processes linking A' to some instance of A ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of))</title>variant</a></text><text x="520.4" y="295.6181678214011" fill="black" class="diagram-class-linkage">[0..*]</text><line y1="316.0" x1="300.4" y2="316.0" x2="180.4" style="stroke:navy;stroke-width:1"/><polygon transform="rotate(180.0 300.4 316.0)" style="fill:navy;stroke:navy;stroke-width:1" points="300.4,316.0 306.4,320.0 312.4,316.0 306.4,312.0 300.4,316.0"/><rect x="225.0" width="30.0" y="309.0" style="fill:white;stroke:black;stroke-width:0" height="18.0"/><text x="240.0" y="319.0" fill="black" class="diagram-class-linkage"><a xlink:href="sequence-definitions.html#Sequence.quality"><title>An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686))</title>quality</a></text><text x="195.4" y="312.0" fill="black" class="diagram-class-linkage">[0..*]</text><line y1="342.7680608365019" x1="476.4" y2="379.2699619771863" x2="596.4" style="stroke:navy;stroke-width:1"/><polygon transform="rotate(16.918843707349374 476.4 342.7680608365019)" style="fill:navy;stroke:navy;stroke-width:1" points="476.4,342.7680608365019 482.4,346.7680608365019 488.4,342.7680608365019 482.4,338.7680608365019 476.4,342.7680608365019"/><rect x="514.0" width="44.0" y="354.0" style="fill:white;stroke:black;stroke-width:0" height="18.0"/><text x="536.0" y="364.0" fill="black" class="diagram-class-linkage"><a xlink:href="sequence-definitions.html#Sequence.repository"><title>Configurations of the external repository</title>repository</a></text><text x="576.4" y="375.2699619771863" fill="black" class="diagram-class-linkage">[0..*]</text><line y1="523.0" x1="307.0" y2="523.0" x2="186.99999999999997" style="stroke:navy;stroke-width:1"/><polygon transform="rotate(180.0 307.0 523.0)" style="fill:navy;stroke:navy;stroke-width:1" points="307.0,523.0 313.0,527.0 319.0,523.0 313.0,519.0 307.0,523.0"/><rect x="236.0" width="22.0" y="516.0" style="fill:white;stroke:black;stroke-width:0" height="18.0"/><text x="247.0" y="526.0" fill="black" class="diagram-class-linkage"><a xlink:href="sequence-definitions.html#Sequence.structureVariant.outer"><title>Structural variant outer</title>outer</a></text><text x="201.99999999999997" y="519.0" fill="black" class="diagram-class-linkage">[0..1]</text><line y1="523.0000000000001" x1="469.8" y2="523.0000000000001" x2="589.8" style="stroke:navy;stroke-width:1"/><polygon transform="rotate(0.0 469.8 523.0000000000001)" style="fill:navy;stroke:navy;stroke-width:1" points="469.8,523.0000000000001 475.8,527.0000000000001 481.8,523.0000000000001 475.8,519.0000000000001 469.8,523.0000000000001"/><rect x="518.0" width="22.0" y="516.0" style="fill:white;stroke:black;stroke-width:0" height="18.0"/><text x="529.0" y="526.0" fill="black" class="diagram-class-linkage"><a xlink:href="sequence-definitions.html#Sequence.structureVariant.inner"><title>Structural variant inner</title>inner</a></text><text x="569.8" y="519.0000000000001" fill="black" class="diagram-class-linkage">[0..1]</text><line y1="403.99999999999994" x1="388.4" y2="484.00000000000006" x2="388.4" style="stroke:navy;stroke-width:1"/><polygon transform="rotate(90.0 388.4 403.99999999999994)" style="fill:navy;stroke:navy;stroke-width:1" points="388.4,403.99999999999994 394.4,407.99999999999994 400.4,403.99999999999994 394.4,399.99999999999994 388.4,403.99999999999994"/><rect x="353.0" width="70.0" y="437.0" style="fill:white;stroke:black;stroke-width:0" height="18.0"/><text x="388.0" y="447.0" fill="black" class="diagram-class-linkage"><a xlink:href="sequence-definitions.html#Sequence.structureVariant"><title>Structural variant</title>structureVariant</a></text><text x="388.4" y="480.00000000000006" fill="black" class="diagram-class-linkage">[0..*]</text></svg>
 </div>
</div>

  </div>
  <div id="tabs-xml">

<div id="xml">
<p><b>XML Template</b></p>
 <div id="xml-inner">
  <pre class="spec">
&lt;<a href="sequence-definitions.html#Sequence" title="Raw data describing a biological sequence." class="dict"><b>Sequence</b></a> xmlns=&quot;http://hl7.org/fhir&quot;&gt; <span style="float: right"><a href="xml.html" title="Documentation for this format"><img src="help.png" alt="doco"/></a></span>
 &lt;!-- from <a href="resource.html">Resource</a>: <a href="resource.html#id">id</a>, <a href="resource.html#meta">meta</a>, <a href="resource.html#implicitRules">implicitRules</a>, and <a href="resource.html#language">language</a> --&gt;
 &lt;!-- from <a href="domainresource.html">DomainResource</a>: <a href="narrative.html#Narrative">text</a>, <a href="references.html#contained">contained</a>, <a href="extensibility.html">extension</a>, and <a href="extensibility.html#modifierExtension">modifierExtension</a> --&gt;
 &lt;<a href="sequence-definitions.html#Sequence.identifier" title="A unique identifier for this particular sequence instance." class="dict"><b>identifier</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: darkgreen"><a href="datatypes.html#Identifier">Identifier</a></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Unique ID for this particular sequence</a></span><span style="color: Gray"> --&gt;</span>&lt;/identifier&gt;
 &lt;<a href="sequence-definitions.html#Sequence.type" title="Amino acid / cDNA transcript / RNA variant." class="dict"><b>type</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="valueset-sequence-type.html">AA | DNA | RNA</a></span><span style="color: Gray"> --&gt;</span>
 &lt;<a href="sequence-definitions.html#Sequence.coordinateSystem" title="Whether the sequence is numbered starting at 0 (0-based numbering or coordinates) or starting at 1 (1-based numbering). Values are &quot;0&quot; for 0-based numbering and &quot;1&quot; for one-based." class="dict"><b>coordinateSystem</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Numbering used for sequence (0-based or 1-based)</a></span><span style="color: Gray"> --&gt;</span>
 &lt;<a href="sequence-definitions.html#Sequence.patient" title="The patient whose sequencing results are described by this resource." class="dict"><b>patient</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="patient.html#Patient">Patient</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Who and/or what this is about</a></span><span style="color: Gray"> --&gt;</span>&lt;/patient&gt;
 &lt;<a href="sequence-definitions.html#Sequence.specimen" title="Specimen used for sequencing." class="dict"><b>specimen</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="specimen.html#Specimen">Specimen</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Specimen used for sequencing</a></span><span style="color: Gray"> --&gt;</span>&lt;/specimen&gt;
 &lt;<a href="sequence-definitions.html#Sequence.device" title="The method for sequencing, for example, chip information." class="dict"><b>device</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="device.html#Device">Device</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">The method for sequencing</a></span><span style="color: Gray"> --&gt;</span>&lt;/device&gt;
 &lt;<a href="sequence-definitions.html#Sequence.quantity" title="Quantity of the sequence." class="dict"><b>quantity</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#Quantity">Quantity</a></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Quantity of the sequence</a></span><span style="color: Gray"> --&gt;</span>&lt;/quantity&gt;
 &lt;<a href="sequence-definitions.html#Sequence.referenceSeq" title="A reference sequence is a sequence that is used to represent an allele or variant." class="dict"><b>referenceSeq</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..1</b></span> Reference sequence --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.chromosome" title="Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340))." class="dict"><b>chromosome</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> <span style="color: navy"><a style="color: navy" href="valueset-chromosome-human.html">Chromosome containing genetic finding</a></span><span style="color: Gray"> --&gt;</span>&lt;/chromosome&gt;
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild" title="The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used." class="dict"><b>genomeBuild</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId" title="Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences." class="dict"><b>referenceSeqId</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> <span style="color: navy"><a style="color: navy" href="valueset-sequence-referenceSeq.html">Reference identifier</a></span><span style="color: Gray"> --&gt;</span>&lt;/referenceSeqId&gt;
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer" title="A Pointer to another Sequence entity as refence sequence." class="dict"><b>referenceSeqPointer</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">A Pointer to another Sequence entity as refence sequence</a></span><span style="color: Gray"> --&gt;</span>&lt;/referenceSeqPointer&gt;
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString" title="A Reference Sequence string." class="dict"><b>referenceSeqString</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">A Reference Sequence string</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.strand" title="Strand of DNA. Available values are &quot;1&quot; for the plus strand and &quot;-1&quot; for the minus strand." class="dict"><b>strand</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Strand of DNA</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.windowStart" title="Start position (inclusive) of the window on the reference sequence." class="dict"><b>windowStart</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Start position (inclusive) of the window on the  reference sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.windowEnd" title="End position (exclusive) of the window on the reference sequence." class="dict"><b>windowEnd</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">End position (exclusive) of the window on the reference sequence</a></span><span style="color: Gray"> --&gt;</span>
 &lt;/referenceSeq&gt;
 &lt;<a href="sequence-definitions.html#Sequence.variant" title="A' is a variant (mutation) of A = definition every instance of A' is either an immediate mutation of some instance of A, or there is a chain of immediate mutation processes linking A' to some instance of A ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of))." class="dict"><b>variant</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..*</b></span> Sequence variant --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.start" title="Start position (inclusive) of the variant on the  reference sequence." class="dict"><b>start</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Start position (inclusive) of the variant on the  reference sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.end" title="End position (exclusive) of the variant on the reference sequence." class="dict"><b>end</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">End position (exclusive) of the variant on the reference sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.observedAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand." class="dict"><b>observedAllele</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Allele that was observed</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.referenceAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand." class="dict"><b>referenceAllele</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Allele of reference sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.cigar" title="Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm)." class="dict"><b>cigar</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Extended CIGAR string for aligning the sequence with reference bases</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.variantPointer" title="A pointer to an Observation containing variant information." class="dict"><b>variantPointer</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="observation.html#Observation">Observation</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Pointer to observed variant information</a></span><span style="color: Gray"> --&gt;</span>&lt;/variantPointer&gt;
 &lt;/variant&gt;
 &lt;<a href="sequence-definitions.html#Sequence.observedSeq" title="Sequence that was observed." class="dict"><b>observedSeq</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Observed sequence</a></span><span style="color: Gray"> --&gt;</span>
 &lt;<a href="sequence-definitions.html#Sequence.quality" title="An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686))." class="dict"><b>quality</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..*</b></span> Sequence quality --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.standardSequence" title="Gold standard sequence used for comparing against." class="dict"><b>standardSequence</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Standard sequence for comparison</a></span><span style="color: Gray"> --&gt;</span>&lt;/standardSequence&gt;
  &lt;<a href="sequence-definitions.html#Sequence.quality.start" title="Start position (inclusive) of the sequence." class="dict"><b>start</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Start position (inclusive) of the sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.end" title="End position (exclusive) of the sequence." class="dict"><b>end</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">End position (exclusive) of the sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.score" title="The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685))." class="dict"><b>score</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#Quantity">Quantity</a></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Quality score</a></span><span style="color: Gray"> --&gt;</span>&lt;/score&gt;
  &lt;<a href="sequence-definitions.html#Sequence.quality.method" title="Method for quality." class="dict"><b>method</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Method for quality</a></span><span style="color: Gray"> --&gt;</span>&lt;/method&gt;
  &lt;<a href="sequence-definitions.html#Sequence.quality.truthTP" title="True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict"><b>truthTP</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">True positives from the perspective of the truth data</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.queryTP" title="True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict"><b>queryTP</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">True positives from the perspective of the query data</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.truthFN" title="False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here." class="dict"><b>truthFN</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">False negatives</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.queryFP" title="False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here." class="dict"><b>queryFP</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">False positives</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.gtFP" title="The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar)." class="dict"><b>gtFP</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">False positives where the non-REF alleles in the Truth and Query Call Sets match</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.precision" title="QUERY.TP / (QUERY.TP + QUERY.FP)." class="dict"><b>precision</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Precision</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.recall" title="TRUTH.TP / (TRUTH.TP + TRUTH.FN)." class="dict"><b>recall</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Recall</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.fScore" title="Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall)." class="dict"><b>fScore</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">F-score</a></span><span style="color: Gray"> --&gt;</span>
 &lt;/quality&gt;
 &lt;<a href="sequence-definitions.html#Sequence.readCoverage" title="Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence." class="dict"><b>readCoverage</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Average number of reads representing a given nucleotide in the reconstructed sequence</a></span><span style="color: Gray"> --&gt;</span>
 &lt;<a href="sequence-definitions.html#Sequence.repository" title="Configurations of the external repository." class="dict"><b>repository</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..*</b></span> External repository --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.url" title="URI of an external repository which contains further details about the genetics data." class="dict"><b>url</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#uri">uri</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">URI of the repository</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.name" title="URI of an external repository which contains further details about the genetics data." class="dict"><b>name</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Name of the repository</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.variantId" title="Id of the variant in this external repository." class="dict"><b>variantId</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Id of the variant</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.readId" title="Id of the read in this external repository." class="dict"><b>readId</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Id of the read</a></span><span style="color: Gray"> --&gt;</span>
 &lt;/repository&gt;
 &lt;<a href="sequence-definitions.html#Sequence.pointer" title="Pointer to next atomic sequence which at most contains one variant." class="dict"><b>pointer</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Pointer to next atomic sequence</a></span><span style="color: Gray"> --&gt;</span>&lt;/pointer&gt;
 &lt;<a href="sequence-definitions.html#Sequence.structureVariant" title="Structural variant." class="dict"><b>structureVariant</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..*</b></span> Structural variant --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.precisionOfBoundaries" title="Precision of boundaries." class="dict"><b>precisionOfBoundaries</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Precision of boundaries</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio" title="Structural Variant reported aCGH ratio." class="dict"><b>reportedaCGHRatio</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant reported aCGH ratio</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.length" title="Structural Variant Length." class="dict"><b>length</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Length</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.outer" title="Structural variant outer." class="dict"><b>outer</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..1</b></span> Structural variant outer --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.structureVariant.outer.start" title="Structural Variant Outer Start-End." class="dict"><b>start</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Outer Start-End</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.structureVariant.outer.end" title="Structural Variant Outer Start-End." class="dict"><b>end</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Outer Start-End</a></span><span style="color: Gray"> --&gt;</span>
  &lt;/outer&gt;
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.inner" title="Structural variant inner." class="dict"><b>inner</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..1</b></span> Structural variant inner --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.structureVariant.inner.start" title="Structural Variant Inner Start-End." class="dict"><b>start</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Inner Start-End</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.structureVariant.inner.end" title="Structural Variant Inner Start-End." class="dict"><b>end</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Inner Start-End</a></span><span style="color: Gray"> --&gt;</span>
  &lt;/inner&gt;
 &lt;/structureVariant&gt;
&lt;/Sequence&gt;
</pre>

 </div>
</div>

  </div>
  <div id="tabs-json">

<div id="json">
<p><b>JSON Template</b></p>
 <div id="json-inner">
  <pre class="spec">
{<span style="float: right"><a href="json.html" title="Documentation for this format"><img src="help.png" alt="doco"/></a></span>
  &quot;resourceType&quot; : &quot;<a href="sequence-definitions.html#Sequence" title="Raw data describing a biological sequence." class="dict"><b>Sequence</b></a>&quot;,
  // from <a href="resource.html">Resource</a>: <a href="resource.html#id">id</a>, <a href="resource.html#meta">meta</a>, <a href="resource.html#implicitRules">implicitRules</a>, and <a href="resource.html#language">language</a>
  // from <a href="domainresource.html">DomainResource</a>: <a href="narrative.html#Narrative">text</a>, <a href="references.html#contained">contained</a>, <a href="extensibility.html">extension</a>, and <a href="extensibility.html#modifierExtension">modifierExtension</a>
  &quot;<a href="sequence-definitions.html#Sequence.identifier" title="A unique identifier for this particular sequence instance." class="dict">identifier</a>&quot; : [{ <span style="color: darkgreen"><a href="datatypes.html#Identifier">Identifier</a></span> }], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Unique ID for this particular sequence</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.type" title="Amino acid / cDNA transcript / RNA variant." class="dict">type</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-sequence-type.html">AA | DNA | RNA</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.coordinateSystem" title="Whether the sequence is numbered starting at 0 (0-based numbering or coordinates) or starting at 1 (1-based numbering). Values are &quot;0&quot; for 0-based numbering and &quot;1&quot; for one-based." class="dict">coordinateSystem</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Numbering used for sequence (0-based or 1-based)</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.patient" title="The patient whose sequencing results are described by this resource." class="dict">patient</a>&quot; : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="patient.html#Patient">Patient</a>)</span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Who and/or what this is about</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.specimen" title="Specimen used for sequencing." class="dict">specimen</a>&quot; : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="specimen.html#Specimen">Specimen</a>)</span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Specimen used for sequencing</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.device" title="The method for sequencing, for example, chip information." class="dict">device</a>&quot; : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="device.html#Device">Device</a>)</span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">The method for sequencing</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.quantity" title="Quantity of the sequence." class="dict">quantity</a>&quot; : { <span style="color: darkgreen"><a href="datatypes.html#Quantity">Quantity</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Quantity of the sequence</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.referenceSeq" title="A reference sequence is a sequence that is used to represent an allele or variant." class="dict">referenceSeq</a>&quot; : { <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Reference sequence</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.referenceSeq.chromosome" title="Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340))." class="dict">chromosome</a>&quot; : { <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-chromosome-human.html">Chromosome containing genetic finding</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild" title="The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used." class="dict">genomeBuild</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId" title="Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences." class="dict">referenceSeqId</a>&quot; : { <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> }, <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-sequence-referenceSeq.html">Reference identifier</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer" title="A Pointer to another Sequence entity as refence sequence." class="dict">referenceSeqPointer</a>&quot; : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>)</span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">A Pointer to another Sequence entity as refence sequence</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString" title="A Reference Sequence string." class="dict">referenceSeqString</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">A Reference Sequence string</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.referenceSeq.strand" title="Strand of DNA. Available values are &quot;1&quot; for the plus strand and &quot;-1&quot; for the minus strand." class="dict">strand</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Strand of DNA</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.referenceSeq.windowStart" title="Start position (inclusive) of the window on the reference sequence." class="dict">windowStart</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Start position (inclusive) of the window on the  reference sequence</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.referenceSeq.windowEnd" title="End position (exclusive) of the window on the reference sequence." class="dict">windowEnd</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt; <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">End position (exclusive) of the window on the reference sequence</a></span>
  },
  &quot;<a href="sequence-definitions.html#Sequence.variant" title="A' is a variant (mutation) of A = definition every instance of A' is either an immediate mutation of some instance of A, or there is a chain of immediate mutation processes linking A' to some instance of A ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of))." class="dict">variant</a>&quot; : [{ <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Sequence variant</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.variant.start" title="Start position (inclusive) of the variant on the  reference sequence." class="dict">start</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Start position (inclusive) of the variant on the  reference sequence</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.variant.end" title="End position (exclusive) of the variant on the reference sequence." class="dict">end</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">End position (exclusive) of the variant on the reference sequence</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.variant.observedAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand." class="dict">observedAllele</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Allele that was observed</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.variant.referenceAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand." class="dict">referenceAllele</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Allele of reference sequence</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.variant.cigar" title="Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm)." class="dict">cigar</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Extended CIGAR string for aligning the sequence with reference bases</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.variant.variantPointer" title="A pointer to an Observation containing variant information." class="dict">variantPointer</a>&quot; : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="observation.html#Observation">Observation</a>)</span> } <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Pointer to observed variant information</a></span>
  }],
  &quot;<a href="sequence-definitions.html#Sequence.observedSeq" title="Sequence that was observed." class="dict">observedSeq</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Observed sequence</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.quality" title="An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686))." class="dict">quality</a>&quot; : [{ <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Sequence quality</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.standardSequence" title="Gold standard sequence used for comparing against." class="dict">standardSequence</a>&quot; : { <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Standard sequence for comparison</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.start" title="Start position (inclusive) of the sequence." class="dict">start</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Start position (inclusive) of the sequence</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.end" title="End position (exclusive) of the sequence." class="dict">end</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">End position (exclusive) of the sequence</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.score" title="The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685))." class="dict">score</a>&quot; : { <span style="color: darkgreen"><a href="datatypes.html#Quantity">Quantity</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Quality score</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.method" title="Method for quality." class="dict">method</a>&quot; : { <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Method for quality</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.truthTP" title="True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict">truthTP</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">True positives from the perspective of the truth data</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.queryTP" title="True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict">queryTP</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">True positives from the perspective of the query data</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.truthFN" title="False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here." class="dict">truthFN</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">False negatives</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.queryFP" title="False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here." class="dict">queryFP</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">False positives</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.gtFP" title="The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar)." class="dict">gtFP</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">False positives where the non-REF alleles in the Truth and Query Call Sets match</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.precision" title="QUERY.TP / (QUERY.TP + QUERY.FP)." class="dict">precision</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Precision</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.recall" title="TRUTH.TP / (TRUTH.TP + TRUTH.FN)." class="dict">recall</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Recall</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.fScore" title="Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall)." class="dict">fScore</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt; <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">F-score</a></span>
  }],
  &quot;<a href="sequence-definitions.html#Sequence.readCoverage" title="Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence." class="dict">readCoverage</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Average number of reads representing a given nucleotide in the reconstructed sequence</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.repository" title="Configurations of the external repository." class="dict">repository</a>&quot; : [{ <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">External repository</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.repository.url" title="URI of an external repository which contains further details about the genetics data." class="dict">url</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#uri">uri</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">URI of the repository</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.repository.name" title="URI of an external repository which contains further details about the genetics data." class="dict">name</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Name of the repository</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.repository.variantId" title="Id of the variant in this external repository." class="dict">variantId</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Id of the variant</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.repository.readId" title="Id of the read in this external repository." class="dict">readId</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot; <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Id of the read</a></span>
  }],
  &quot;<a href="sequence-definitions.html#Sequence.pointer" title="Pointer to next atomic sequence which at most contains one variant." class="dict">pointer</a>&quot; : [{ <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>)</span> }], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Pointer to next atomic sequence</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.structureVariant" title="Structural variant." class="dict">structureVariant</a>&quot; : [{ <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural variant</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.structureVariant.precisionOfBoundaries" title="Precision of boundaries." class="dict">precisionOfBoundaries</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Precision of boundaries</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio" title="Structural Variant reported aCGH ratio." class="dict">reportedaCGHRatio</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant reported aCGH ratio</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.structureVariant.length" title="Structural Variant Length." class="dict">length</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Length</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.structureVariant.outer" title="Structural variant outer." class="dict">outer</a>&quot; : { <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural variant outer</a></span>
      &quot;<a href="sequence-definitions.html#Sequence.structureVariant.outer.start" title="Structural Variant Outer Start-End." class="dict">start</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Outer Start-End</a></span>
      &quot;<a href="sequence-definitions.html#Sequence.structureVariant.outer.end" title="Structural Variant Outer Start-End." class="dict">end</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt; <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Outer Start-End</a></span>
    },
    &quot;<a href="sequence-definitions.html#Sequence.structureVariant.inner" title="Structural variant inner." class="dict">inner</a>&quot; : { <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural variant inner</a></span>
      &quot;<a href="sequence-definitions.html#Sequence.structureVariant.inner.start" title="Structural Variant Inner Start-End." class="dict">start</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Inner Start-End</a></span>
      &quot;<a href="sequence-definitions.html#Sequence.structureVariant.inner.end" title="Structural Variant Inner Start-End." class="dict">end</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt; <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Inner Start-End</a></span>
    }
  }]
}
</pre>

 </div>
</div>

  </div>
  <div id="tabs-ttl">

<div id="ttl">
<p><b>Turtle Template</b></p>
 <div id="json-inner">
  <pre class="spec">
@prefix fhir: &lt;http://hl7.org/fhir/&gt; .<span style="float: right"><a href="rdf.html" title="Documentation for this format"><img src="help.png" alt="doco"/></a></span>


[ a fhir:<a href="sequence-definitions.html#Sequence" title="Raw data describing a biological sequence." class="dict"><b>Sequence</b></a>;
  fhir:nodeRole fhir:treeRoot; # if this is the parser root

  # from <a href="resource.html">Resource</a>: <a href="resource.html#id">.id</a>, <a href="resource.html#meta">.meta</a>, <a href="resource.html#implicitRules">.implicitRules</a>, and <a href="resource.html#language">.language</a>
  # from <a href="domainresource.html">DomainResource</a>: <a href="narrative.html#Narrative">.text</a>, <a href="references.html#contained">.contained</a>, <a href="extensibility.html">.extension</a>, and <a href="extensibility.html#modifierExtension">.modifierExtension</a>
  fhir:<a href="sequence-definitions.html#Sequence.identifier" title="A unique identifier for this particular sequence instance." class="dict">Sequence.identifier</a><a name="ttl-identifier"> </a>[ <a href="datatypes.html#Identifier">Identifier</a> ], ... ; # 0..* <span style="color: navy">Unique ID for this particular sequence</span>
  fhir:<a href="sequence-definitions.html#Sequence.type" title="Amino acid / cDNA transcript / RNA variant." class="dict">Sequence.type</a><a name="ttl-type"> </a>[ <a href="datatypes.html#code">code</a> ]; # 1..1 <span style="color: navy">AA | DNA | RNA</span>
  fhir:<a href="sequence-definitions.html#Sequence.coordinateSystem" title="Whether the sequence is numbered starting at 0 (0-based numbering or coordinates) or starting at 1 (1-based numbering). Values are &quot;0&quot; for 0-based numbering and &quot;1&quot; for one-based." class="dict">Sequence.coordinateSystem</a><a name="ttl-coordinateSystem"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 1..1 <span style="color: navy">Numbering used for sequence (0-based or 1-based)</span>
  fhir:<a href="sequence-definitions.html#Sequence.patient" title="The patient whose sequencing results are described by this resource." class="dict">Sequence.patient</a><a name="ttl-patient"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="patient.html#Patient">Patient</a>) ]; # 0..1 <span style="color: navy">Who and/or what this is about</span>
  fhir:<a href="sequence-definitions.html#Sequence.specimen" title="Specimen used for sequencing." class="dict">Sequence.specimen</a><a name="ttl-specimen"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="specimen.html#Specimen">Specimen</a>) ]; # 0..1 <span style="color: navy">Specimen used for sequencing</span>
  fhir:<a href="sequence-definitions.html#Sequence.device" title="The method for sequencing, for example, chip information." class="dict">Sequence.device</a><a name="ttl-device"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="device.html#Device">Device</a>) ]; # 0..1 <span style="color: navy">The method for sequencing</span>
  fhir:<a href="sequence-definitions.html#Sequence.quantity" title="Quantity of the sequence." class="dict">Sequence.quantity</a><a name="ttl-quantity"> </a>[ <a href="datatypes.html#Quantity">Quantity</a> ]; # 0..1 <span style="color: navy">Quantity of the sequence</span>
  fhir:<a href="sequence-definitions.html#Sequence.referenceSeq" title="A reference sequence is a sequence that is used to represent an allele or variant." class="dict">Sequence.referenceSeq</a><a name="ttl-referenceSeq"> </a>[ # 0..1 <span style="color: navy">Reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.chromosome" title="Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340))." class="dict">Sequence.referenceSeq.chromosome</a><a name="ttl-chromosome"> </a>[ <a href="datatypes.html#CodeableConcept">CodeableConcept</a> ]; # 0..1 <span style="color: navy">Chromosome containing genetic finding</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild" title="The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used." class="dict">Sequence.referenceSeq.genomeBuild</a><a name="ttl-genomeBuild"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId" title="Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences." class="dict">Sequence.referenceSeq.referenceSeqId</a><a name="ttl-referenceSeqId"> </a>[ <a href="datatypes.html#CodeableConcept">CodeableConcept</a> ]; # 1..1 <span style="color: navy">Reference identifier</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer" title="A Pointer to another Sequence entity as refence sequence." class="dict">Sequence.referenceSeq.referenceSeqPointer</a><a name="ttl-referenceSeqPointer"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>) ]; # 0..1 <span style="color: navy">A Pointer to another Sequence entity as refence sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString" title="A Reference Sequence string." class="dict">Sequence.referenceSeq.referenceSeqString</a><a name="ttl-referenceSeqString"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">A Reference Sequence string</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.strand" title="Strand of DNA. Available values are &quot;1&quot; for the plus strand and &quot;-1&quot; for the minus strand." class="dict">Sequence.referenceSeq.strand</a><a name="ttl-strand"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 1..1 <span style="color: navy">Strand of DNA</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.windowStart" title="Start position (inclusive) of the window on the reference sequence." class="dict">Sequence.referenceSeq.windowStart</a><a name="ttl-windowStart"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 1..1 <span style="color: navy">Start position (inclusive) of the window on the  reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.windowEnd" title="End position (exclusive) of the window on the reference sequence." class="dict">Sequence.referenceSeq.windowEnd</a><a name="ttl-windowEnd"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 1..1 <span style="color: navy">End position (exclusive) of the window on the reference sequence</span>
  ];
  fhir:<a href="sequence-definitions.html#Sequence.variant" title="A' is a variant (mutation) of A = definition every instance of A' is either an immediate mutation of some instance of A, or there is a chain of immediate mutation processes linking A' to some instance of A ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of))." class="dict">Sequence.variant</a><a name="ttl-variant"> </a>[ # 0..* <span style="color: navy">Sequence variant</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.start" title="Start position (inclusive) of the variant on the  reference sequence." class="dict">Sequence.variant.start</a><a name="ttl-start"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Start position (inclusive) of the variant on the  reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.end" title="End position (exclusive) of the variant on the reference sequence." class="dict">Sequence.variant.end</a><a name="ttl-end"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">End position (exclusive) of the variant on the reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.observedAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand." class="dict">Sequence.variant.observedAllele</a><a name="ttl-observedAllele"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Allele that was observed</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.referenceAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand." class="dict">Sequence.variant.referenceAllele</a><a name="ttl-referenceAllele"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Allele of reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.cigar" title="Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm)." class="dict">Sequence.variant.cigar</a><a name="ttl-cigar"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Extended CIGAR string for aligning the sequence with reference bases</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.variantPointer" title="A pointer to an Observation containing variant information." class="dict">Sequence.variant.variantPointer</a><a name="ttl-variantPointer"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="observation.html#Observation">Observation</a>) ]; # 0..1 <span style="color: navy">Pointer to observed variant information</span>
  ], ...;
  fhir:<a href="sequence-definitions.html#Sequence.observedSeq" title="Sequence that was observed." class="dict">Sequence.observedSeq</a><a name="ttl-observedSeq"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Observed sequence</span>
  fhir:<a href="sequence-definitions.html#Sequence.quality" title="An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686))." class="dict">Sequence.quality</a><a name="ttl-quality"> </a>[ # 0..* <span style="color: navy">Sequence quality</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.standardSequence" title="Gold standard sequence used for comparing against." class="dict">Sequence.quality.standardSequence</a><a name="ttl-standardSequence"> </a>[ <a href="datatypes.html#CodeableConcept">CodeableConcept</a> ]; # 0..1 <span style="color: navy">Standard sequence for comparison</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.start" title="Start position (inclusive) of the sequence." class="dict">Sequence.quality.start</a><a name="ttl-start"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Start position (inclusive) of the sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.end" title="End position (exclusive) of the sequence." class="dict">Sequence.quality.end</a><a name="ttl-end"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">End position (exclusive) of the sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.score" title="The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685))." class="dict">Sequence.quality.score</a><a name="ttl-score"> </a>[ <a href="datatypes.html#Quantity">Quantity</a> ]; # 0..1 <span style="color: navy">Quality score</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.method" title="Method for quality." class="dict">Sequence.quality.method</a><a name="ttl-method"> </a>[ <a href="datatypes.html#CodeableConcept">CodeableConcept</a> ]; # 0..1 <span style="color: navy">Method for quality</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.truthTP" title="True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict">Sequence.quality.truthTP</a><a name="ttl-truthTP"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">True positives from the perspective of the truth data</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.queryTP" title="True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict">Sequence.quality.queryTP</a><a name="ttl-queryTP"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">True positives from the perspective of the query data</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.truthFN" title="False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here." class="dict">Sequence.quality.truthFN</a><a name="ttl-truthFN"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">False negatives</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.queryFP" title="False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here." class="dict">Sequence.quality.queryFP</a><a name="ttl-queryFP"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">False positives</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.gtFP" title="The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar)." class="dict">Sequence.quality.gtFP</a><a name="ttl-gtFP"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">False positives where the non-REF alleles in the Truth and Query Call Sets match</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.precision" title="QUERY.TP / (QUERY.TP + QUERY.FP)." class="dict">Sequence.quality.precision</a><a name="ttl-precision"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">Precision</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.recall" title="TRUTH.TP / (TRUTH.TP + TRUTH.FN)." class="dict">Sequence.quality.recall</a><a name="ttl-recall"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">Recall</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.fScore" title="Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall)." class="dict">Sequence.quality.fScore</a><a name="ttl-fScore"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">F-score</span>
  ], ...;
  fhir:<a href="sequence-definitions.html#Sequence.readCoverage" title="Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence." class="dict">Sequence.readCoverage</a><a name="ttl-readCoverage"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Average number of reads representing a given nucleotide in the reconstructed sequence</span>
  fhir:<a href="sequence-definitions.html#Sequence.repository" title="Configurations of the external repository." class="dict">Sequence.repository</a><a name="ttl-repository"> </a>[ # 0..* <span style="color: navy">External repository</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.url" title="URI of an external repository which contains further details about the genetics data." class="dict">Sequence.repository.url</a><a name="ttl-url"> </a>[ <a href="datatypes.html#uri">uri</a> ]; # 0..1 <span style="color: navy">URI of the repository</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.name" title="URI of an external repository which contains further details about the genetics data." class="dict">Sequence.repository.name</a><a name="ttl-name"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Name of the repository</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.variantId" title="Id of the variant in this external repository." class="dict">Sequence.repository.variantId</a><a name="ttl-variantId"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Id of the variant</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.readId" title="Id of the read in this external repository." class="dict">Sequence.repository.readId</a><a name="ttl-readId"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Id of the read</span>
  ], ...;
  fhir:<a href="sequence-definitions.html#Sequence.pointer" title="Pointer to next atomic sequence which at most contains one variant." class="dict">Sequence.pointer</a><a name="ttl-pointer"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>) ], ... ; # 0..* <span style="color: navy">Pointer to next atomic sequence</span>
  fhir:<a href="sequence-definitions.html#Sequence.structureVariant" title="Structural variant." class="dict">Sequence.structureVariant</a><a name="ttl-structureVariant"> </a>[ # 0..* <span style="color: navy">Structural variant</span>
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.precisionOfBoundaries" title="Precision of boundaries." class="dict">Sequence.structureVariant.precisionOfBoundaries</a><a name="ttl-precisionOfBoundaries"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Precision of boundaries</span>
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio" title="Structural Variant reported aCGH ratio." class="dict">Sequence.structureVariant.reportedaCGHRatio</a><a name="ttl-reportedaCGHRatio"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">Structural Variant reported aCGH ratio</span>
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.length" title="Structural Variant Length." class="dict">Sequence.structureVariant.length</a><a name="ttl-length"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Length</span>
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.outer" title="Structural variant outer." class="dict">Sequence.structureVariant.outer</a><a name="ttl-outer"> </a>[ # 0..1 <span style="color: navy">Structural variant outer</span>
      fhir:<a href="sequence-definitions.html#Sequence.structureVariant.outer.start" title="Structural Variant Outer Start-End." class="dict">Sequence.structureVariant.outer.start</a><a name="ttl-start"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Outer Start-End</span>
      fhir:<a href="sequence-definitions.html#Sequence.structureVariant.outer.end" title="Structural Variant Outer Start-End." class="dict">Sequence.structureVariant.outer.end</a><a name="ttl-end"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Outer Start-End</span>
    ];
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.inner" title="Structural variant inner." class="dict">Sequence.structureVariant.inner</a><a name="ttl-inner"> </a>[ # 0..1 <span style="color: navy">Structural variant inner</span>
      fhir:<a href="sequence-definitions.html#Sequence.structureVariant.inner.start" title="Structural Variant Inner Start-End." class="dict">Sequence.structureVariant.inner.start</a><a name="ttl-start"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Inner Start-End</span>
      fhir:<a href="sequence-definitions.html#Sequence.structureVariant.inner.end" title="Structural Variant Inner Start-End." class="dict">Sequence.structureVariant.inner.end</a><a name="ttl-end"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Inner Start-End</span>
    ];
  ], ...;
]
</pre>

 </div>
</div>

  </div>
  <div id="tabs-diff">

<div id="vdiff">
<p><b>Changes since DSTU2</b></p>
 <div id="diff-inner">
  <p>This resource did not exist in Release 2</p>
 </div>
</div>

  </div>
  <div id="tabs-all">

<div id="tbl">
<a name="tbl"> </a>
<p><b>Structure</b></p>
 <div id="tbl-inner">
  <table border="0" cellpadding="0" cellspacing="0" style="border: 0px; font-size: 11px; font-family: verdana; vertical-align: top;"><tr style="border: 1px #F0F0F0 solid; font-size: 11px; font-family: verdana; vertical-align: top;"><th style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="formats.html#table" title="The logical name of the element">Name</a></th><th style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="formats.html#table" title="Information about the use of the element">Flags</a></th><th style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="formats.html#table" title="Minimum and Maximum # of times the the element can appear in the instance">Card.</a></th><th style="width: 100px" class="hierarchy"><a href="formats.html#table" title="Reference to the type of the element">Type</a></th><th style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="formats.html#table" title="Additional information about the element">Description &amp; Constraints</a><span style="float: right"><a href="formats.html#table" title="Legend for this format"><img src="help16.png" alt="doco" style="background-color: inherit"/></a></span></th></tr><tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck1.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_resource.png" alt="." style="background-color: white; background-color: inherit" title="Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence" title="Sequence : Raw data describing a biological sequence.">Sequence</a><a name="Sequence"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"/><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="domainresource.html">DomainResource</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Information about a biological sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.identifier" title="Sequence.identifier : A unique identifier for this particular sequence instance.">identifier</a><a name="Sequence.identifier"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#Identifier">Identifier</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Unique ID for this particular sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.type" title="Sequence.type : Amino acid / cDNA transcript / RNA variant.">type</a><a name="Sequence.type"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#code">code</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">AA | DNA | RNA<br/><a href="valueset-sequence-type.html" title="Type if a sequence -- DNA, RNA, or amino acid sequence">sequenceType</a> (<a href="terminologies.html#example" title="Instances are not expected or even encouraged to draw from the specified value set.  The value set merely provides examples of the types of concepts intended to be included.">Example</a>)</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.coordinateSystem" title="Sequence.coordinateSystem : Whether the sequence is numbered starting at 0 (0-based numbering or coordinates) or starting at 1 (1-based numbering). Values are &quot;0&quot; for 0-based numbering and &quot;1&quot; for one-based.">coordinateSystem</a><a name="Sequence.coordinateSystem"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Numbering used for sequence (0-based or 1-based)</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.patient" title="Sequence.patient : The patient whose sequencing results are described by this resource.">patient</a><a name="Sequence.patient"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html">Reference</a>(<a href="patient.html">Patient</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Who and/or what this is about</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.specimen" title="Sequence.specimen : Specimen used for sequencing.">specimen</a><a name="Sequence.specimen"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html">Reference</a>(<a href="specimen.html">Specimen</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Specimen used for sequencing</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.device" title="Sequence.device : The method for sequencing, for example, chip information.">device</a><a name="Sequence.device"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html">Reference</a>(<a href="device.html">Device</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">The method for sequencing</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quantity" title="Sequence.quantity : Quantity of the sequence.">quantity</a><a name="Sequence.quantity"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#Quantity">Quantity</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Quantity of the sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck11.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq" title="Sequence.referenceSeq : A reference sequence is a sequence that is used to represent an allele or variant.">referenceSeq</a><a name="Sequence.referenceSeq"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Reference sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.chromosome" title="Sequence.referenceSeq.chromosome : Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340)).">chromosome</a><a name="Sequence.referenceSeq.chromosome"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Chromosome containing genetic finding<br/><a href="valueset-chromosome-human.html" title="Chromosome number for human">chromosome-human</a> (<a href="terminologies.html#example" title="Instances are not expected or even encouraged to draw from the specified value set.  The value set merely provides examples of the types of concepts intended to be included.">Example</a>)</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild" title="Sequence.referenceSeq.genomeBuild : The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used.">genomeBuild</a><a name="Sequence.referenceSeq.genomeBuild"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId" title="Sequence.referenceSeq.referenceSeqId : Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences.">referenceSeqId</a><a name="Sequence.referenceSeq.referenceSeqId"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Reference identifier<br/><a href="valueset-sequence-referenceSeq.html" title="Reference identifier">ENSEMBL</a> (<a href="terminologies.html#example" title="Instances are not expected or even encouraged to draw from the specified value set.  The value set merely provides examples of the types of concepts intended to be included.">Example</a>)</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer" title="Sequence.referenceSeq.referenceSeqPointer : A Pointer to another Sequence entity as refence sequence.">referenceSeqPointer</a><a name="Sequence.referenceSeq.referenceSeqPointer"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html">Reference</a>(<a href="sequence.html">Sequence</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">A Pointer to another Sequence entity as refence sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString" title="Sequence.referenceSeq.referenceSeqString : A Reference Sequence string.">referenceSeqString</a><a name="Sequence.referenceSeq.referenceSeqString"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">A Reference Sequence string</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.strand" title="Sequence.referenceSeq.strand : Strand of DNA. Available values are &quot;1&quot; for the plus strand and &quot;-1&quot; for the minus strand.">strand</a><a name="Sequence.referenceSeq.strand"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Strand of DNA</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.windowStart" title="Sequence.referenceSeq.windowStart : Start position (inclusive) of the window on the reference sequence.">windowStart</a><a name="Sequence.referenceSeq.windowStart"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Start position (inclusive) of the window on the  reference sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck100.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.windowEnd" title="Sequence.referenceSeq.windowEnd : End position (exclusive) of the window on the reference sequence.">windowEnd</a><a name="Sequence.referenceSeq.windowEnd"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">End position (exclusive) of the window on the reference sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck11.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant" title="Sequence.variant : A' is a variant (mutation) of A = definition every instance of A' is either an immediate mutation of some instance of A, or there is a chain of immediate mutation processes linking A' to some instance of A ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of)).">variant</a><a name="Sequence.variant"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Sequence variant</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.start" title="Sequence.variant.start : Start position (inclusive) of the variant on the  reference sequence.">start</a><a name="Sequence.variant.start"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Start position (inclusive) of the variant on the  reference sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.end" title="Sequence.variant.end : End position (exclusive) of the variant on the reference sequence.">end</a><a name="Sequence.variant.end"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">End position (exclusive) of the variant on the reference sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.observedAllele" title="Sequence.variant.observedAllele : An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand.">observedAllele</a><a name="Sequence.variant.observedAllele"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Allele that was observed</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.referenceAllele" title="Sequence.variant.referenceAllele : An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand.">referenceAllele</a><a name="Sequence.variant.referenceAllele"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Allele of reference sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.cigar" title="Sequence.variant.cigar : Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm).">cigar</a><a name="Sequence.variant.cigar"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Extended CIGAR string for aligning the sequence with reference bases</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck100.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.variantPointer" title="Sequence.variant.variantPointer : A pointer to an Observation containing variant information.">variantPointer</a><a name="Sequence.variant.variantPointer"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html">Reference</a>(<a href="observation.html">Observation</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Pointer to observed variant information</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.observedSeq" title="Sequence.observedSeq : Sequence that was observed.">observedSeq</a><a name="Sequence.observedSeq"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Observed sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck11.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality" title="Sequence.quality : An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686)).">quality</a><a name="Sequence.quality"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Sequence quality</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.standardSequence" title="Sequence.quality.standardSequence : Gold standard sequence used for comparing against.">standardSequence</a><a name="Sequence.quality.standardSequence"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Standard sequence for comparison</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.start" title="Sequence.quality.start : Start position (inclusive) of the sequence.">start</a><a name="Sequence.quality.start"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Start position (inclusive) of the sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.end" title="Sequence.quality.end : End position (exclusive) of the sequence.">end</a><a name="Sequence.quality.end"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">End position (exclusive) of the sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.score" title="Sequence.quality.score : The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685)).">score</a><a name="Sequence.quality.score"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#Quantity">Quantity</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Quality score</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.method" title="Sequence.quality.method : Method for quality.">method</a><a name="Sequence.quality.method"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Method for quality</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.truthTP" title="Sequence.quality.truthTP : True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event.">truthTP</a><a name="Sequence.quality.truthTP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">True positives from the perspective of the truth data</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.queryTP" title="Sequence.quality.queryTP : True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event.">queryTP</a><a name="Sequence.quality.queryTP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">True positives from the perspective of the query data</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.truthFN" title="Sequence.quality.truthFN : False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here.">truthFN</a><a name="Sequence.quality.truthFN"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">False negatives</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.queryFP" title="Sequence.quality.queryFP : False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here.">queryFP</a><a name="Sequence.quality.queryFP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">False positives</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.gtFP" title="Sequence.quality.gtFP : The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar).">gtFP</a><a name="Sequence.quality.gtFP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">False positives where the non-REF alleles in the Truth and Query Call Sets match</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.precision" title="Sequence.quality.precision : QUERY.TP / (QUERY.TP + QUERY.FP).">precision</a><a name="Sequence.quality.precision"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Precision</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.recall" title="Sequence.quality.recall : TRUTH.TP / (TRUTH.TP + TRUTH.FN).">recall</a><a name="Sequence.quality.recall"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Recall</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck100.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.fScore" title="Sequence.quality.fScore : Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall).">fScore</a><a name="Sequence.quality.fScore"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">F-score</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.readCoverage" title="Sequence.readCoverage : Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence.">readCoverage</a><a name="Sequence.readCoverage"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Average number of reads representing a given nucleotide in the reconstructed sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck11.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository" title="Sequence.repository : Configurations of the external repository.">repository</a><a name="Sequence.repository"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">External repository</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.url" title="Sequence.repository.url : URI of an external repository which contains further details about the genetics data.">url</a><a name="Sequence.repository.url"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#uri">uri</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">URI of the repository</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.name" title="Sequence.repository.name : URI of an external repository which contains further details about the genetics data.">name</a><a name="Sequence.repository.name"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Name of the repository</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.variantId" title="Sequence.repository.variantId : Id of the variant in this external repository.">variantId</a><a name="Sequence.repository.variantId"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Id of the variant</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck100.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.readId" title="Sequence.repository.readId : Id of the read in this external repository.">readId</a><a name="Sequence.repository.readId"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Id of the read</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.pointer" title="Sequence.pointer : Pointer to next atomic sequence which at most contains one variant.">pointer</a><a name="Sequence.pointer"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html">Reference</a>(<a href="sequence.html">Sequence</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Pointer to next atomic sequence</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck01.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant" title="Sequence.structureVariant : Structural variant.">structureVariant</a><a name="Sequence.structureVariant"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Structural variant</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.precisionOfBoundaries" title="Sequence.structureVariant.precisionOfBoundaries : Precision of boundaries.">precisionOfBoundaries</a><a name="Sequence.structureVariant.precisionOfBoundaries"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Precision of boundaries</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio" title="Sequence.structureVariant.reportedaCGHRatio : Structural Variant reported aCGH ratio.">reportedaCGHRatio</a><a name="Sequence.structureVariant.reportedaCGHRatio"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant reported aCGH ratio</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.length" title="Sequence.structureVariant.length : Structural Variant Length.">length</a><a name="Sequence.structureVariant.length"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Length</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck011.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.outer" title="Sequence.structureVariant.outer : Structural variant outer.">outer</a><a name="Sequence.structureVariant.outer"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Structural variant outer</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck0110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.outer.start" title="Sequence.structureVariant.outer.start : Structural Variant Outer Start-End.">start</a><a name="Sequence.structureVariant.outer.start"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Outer Start-End</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck0100.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.outer.end" title="Sequence.structureVariant.outer.end : Structural Variant Outer Start-End.">end</a><a name="Sequence.structureVariant.outer.end"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Outer Start-End</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck001.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.inner" title="Sequence.structureVariant.inner : Structural variant inner.">inner</a><a name="Sequence.structureVariant.inner"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Structural variant inner</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck0010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.inner.start" title="Sequence.structureVariant.inner.start : Structural Variant Inner Start-End.">start</a><a name="Sequence.structureVariant.inner.start"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Inner Start-End</td></tr>
<tr style="border: 0px; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck0000.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.inner.end" title="Sequence.structureVariant.inner.end : Structural Variant Inner Start-End.">end</a><a name="Sequence.structureVariant.inner.end"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><span style="padding: 3px" title="This element is included in summaries">Σ</span></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Inner Start-End</td></tr>
<tr><td colspan="5" class="hierarchy"><br/><a href="formats.html#table" title="Legend for this format"><img src="help16.png" alt="doco" style="background-color: inherit"/> Documentation for this format</a></td></tr></table>
 </div>
</div>

 
<div id="uml">
<a name="uml"> </a>
<p><b>UML Diagram</b> (<a href="formats.html#uml">Legend</a>)</p>
 <div id="uml-inner">
 <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="986.6" version="1.1" height="585.0">  <defs>
    <filter x="0" width="200%" y="0" id="shadowB" height="200%">
      <feOffset result="offOut" dx="3" in="SourceGraphic" dy="3"/>
      <feColorMatrix result="matrixOut" in="offOut" values="0.2 0 0 0 0 0 0.2 0 0 0 0 0 0.2 0 0 0 0 0 1 0" type="matrix"/>
      <feGaussianBlur result="blurOut" stdDeviation="2" in="matrixOut"/>
      <feBlend mode="normal" in="SourceGraphic" in2="blurOut"/>
    </filter>
  </defs><g><rect filter="url(#shadowB)" rx="4" ry="4" x="300.4" width="176.0" y="228.0" style="fill:#f0f8ff;stroke:black;stroke-width:1" height="176.0"/><line y1="256.0" x1="300.4" y2="256.0" x2="476.4" style="stroke:dimgrey;stroke-width:1"/><text x="388.4" y="248.0" fill="black" class="diagram-class-title  diagram-class-resource">Sequence<tspan class="diagram-class-title-link"> (<a class="diagram-class-reference" xlink:href="domainresource.html">DomainResource</a>)</tspan></text><text x="306.4" y="270.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.identifier"><title>A unique identifier for this particular sequence instance</title>identifier</a> : <a xlink:href="datatypes.html#Identifier">Identifier</a> [0..*]</text><text x="306.4" y="284.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.type"><title>Amino acid / cDNA transcript / RNA variant</title>type</a> : <a xlink:href="datatypes.html#code">code</a> [1..1] « <a xlink:href="valueset-sequence-type.html"><title>Type if a sequence -- DNA, RNA, or amino acid sequence (Strength=Example)</title>sequenceType</a>?? »</text><text x="306.4" y="298.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.coordinateSystem"><title>Whether the sequence is numbered starting at 0 (0-based numbering or coordinates) or starting at 1 (1-based numbering). Values are &quot;0&quot; for 0-based numbering and &quot;1&quot; for one-based</title>coordinateSystem</a> : <a xlink:href="datatypes.html#integer">integer</a> [1..1]</text><text x="306.4" y="312.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.patient"><title>The patient whose sequencing results are described by this resource</title>patient</a> : <a xlink:href="references.html#Reference">Reference</a> [0..1] « <a xlink:href="patient.html#Patient">Patient</a> »</text><text x="306.4" y="326.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.specimen"><title>Specimen used for sequencing</title>specimen</a> : <a xlink:href="references.html#Reference">Reference</a> [0..1] « <a xlink:href="specimen.html#Specimen">Specimen</a> »</text><text x="306.4" y="340.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.device"><title>The method for sequencing, for example, chip information</title>device</a> : <a xlink:href="references.html#Reference">Reference</a> [0..1] « <a xlink:href="device.html#Device">Device</a> »</text><text x="306.4" y="354.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quantity"><title>Quantity of the sequence</title>quantity</a> : <a xlink:href="datatypes.html#Quantity">Quantity</a> [0..1]</text><text x="306.4" y="368.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.observedSeq"><title>Sequence that was observed</title>observedSeq</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="306.4" y="382.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.readCoverage"><title>Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence</title>readCoverage</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="306.4" y="396.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.pointer"><title>Pointer to next atomic sequence which at most contains one variant</title>pointer</a> : <a xlink:href="references.html#Reference">Reference</a> [0..*] « <a xlink:href="sequence.html#Sequence">Sequence</a> »</text><g><rect filter="url(#shadowB)" rx="4" ry="4" x="265.19999999999993" width="246.40000000000003" y="0.0" style="fill:#f0f8ff;stroke:black;stroke-width:1" height="148.0"/><line y1="28.0" x1="265.19999999999993" y2="28.0" x2="511.59999999999997" style="stroke:dimgrey;stroke-width:1"/><text x="388.4" y="20.0" fill="black" class="diagram-class-title">ReferenceSeq</text><text x="271.19999999999993" y="42.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.referenceSeq.chromosome"><title>Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340))</title>chromosome</a> : <a xlink:href="datatypes.html#CodeableConcept">CodeableConcept</a> [0..1] « <a xlink:href="valueset-chromosome-human.html"><title>Chromosome number for human (Strength=Example)</title>chromosome-human</a>?? »</text><text x="271.19999999999993" y="56.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild"><title>The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used</title>genomeBuild</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="271.19999999999993" y="70.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId"><title>Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences</title>referenceSeqId</a> : <a xlink:href="datatypes.html#CodeableConcept">CodeableConcept</a> [1..1] « <a xlink:href="valueset-sequence-referenceSeq.html"><title>Reference identifier (Strength=Example)</title>ENSEMBL</a>?? »</text><text x="271.19999999999993" y="84.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer"><title>A Pointer to another Sequence entity as refence sequence</title>referenceSeqPointer</a> : <a xlink:href="references.html#Reference">Reference</a> [0..1] « <a xlink:href="sequence.html#Sequence">Sequence</a> »</text><text x="271.19999999999993" y="98.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString"><title>A Reference Sequence string</title>referenceSeqString</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="271.19999999999993" y="112.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.referenceSeq.strand"><title>Strand of DNA. Available values are &quot;1&quot; for the plus strand and &quot;-1&quot; for the minus strand</title>strand</a> : <a xlink:href="datatypes.html#integer">integer</a> [1..1]</text><text x="271.19999999999993" y="126.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.referenceSeq.windowStart"><title>Start position (inclusive) of the window on the reference sequence</title>windowStart</a> : <a xlink:href="datatypes.html#integer">integer</a> [1..1]</text><text x="271.19999999999993" y="140.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.referenceSeq.windowEnd"><title>End position (exclusive) of the window on the reference sequence</title>windowEnd</a> : <a xlink:href="datatypes.html#integer">integer</a> [1..1]</text></g><g><rect filter="url(#shadowB)" rx="4" ry="4" x="540.4" width="215.60000000000002" y="228.0" style="fill:#f0f8ff;stroke:black;stroke-width:1" height="120.0"/><line y1="256.0" x1="540.4" y2="256.0" x2="756.0" style="stroke:dimgrey;stroke-width:1"/><text x="648.2" y="248.0" fill="black" class="diagram-class-title">Variant</text><text x="546.4" y="270.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.variant.start"><title>Start position (inclusive) of the variant on the  reference sequence</title>start</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="546.4" y="284.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.variant.end"><title>End position (exclusive) of the variant on the reference sequence</title>end</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="546.4" y="298.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.variant.observedAllele"><title>An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand</title>observedAllele</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="546.4" y="312.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.variant.referenceAllele"><title>An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand</title>referenceAllele</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="546.4" y="326.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.variant.cigar"><title>Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm)</title>cigar</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="546.4" y="340.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.variant.variantPointer"><title>A pointer to an Observation containing variant information</title>variantPointer</a> : <a xlink:href="references.html#Reference">Reference</a> [0..1] « <a xlink:href="observation.html#Observation">Observation</a> »</text></g><g><rect filter="url(#shadowB)" rx="4" ry="4" x="0.0" width="180.4" y="207.0" style="fill:#f0f8ff;stroke:black;stroke-width:1" height="218.0"/><line y1="235.0" x1="0.0" y2="235.0" x2="180.4" style="stroke:dimgrey;stroke-width:1"/><text x="90.2" y="227.0" fill="black" class="diagram-class-title">Quality</text><text x="6.0" y="249.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.standardSequence"><title>Gold standard sequence used for comparing against</title>standardSequence</a> : <a xlink:href="datatypes.html#CodeableConcept">CodeableConcept</a> [0..1]</text><text x="6.0" y="263.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.start"><title>Start position (inclusive) of the sequence</title>start</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="6.0" y="277.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.end"><title>End position (exclusive) of the sequence</title>end</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="6.0" y="291.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.score"><title>The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685))</title>score</a> : <a xlink:href="datatypes.html#Quantity">Quantity</a> [0..1]</text><text x="6.0" y="305.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.method"><title>Method for quality</title>method</a> : <a xlink:href="datatypes.html#CodeableConcept">CodeableConcept</a> [0..1]</text><text x="6.0" y="319.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.truthTP"><title>True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event</title>truthTP</a> : <a xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="333.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.queryTP"><title>True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event</title>queryTP</a> : <a xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="347.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.truthFN"><title>False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here</title>truthFN</a> : <a xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="361.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.queryFP"><title>False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here</title>queryFP</a> : <a xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="375.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.gtFP"><title>The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar)</title>gtFP</a> : <a xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="389.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.precision"><title>QUERY.TP / (QUERY.TP + QUERY.FP)</title>precision</a> : <a xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="403.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.recall"><title>TRUTH.TP / (TRUTH.TP + TRUTH.FN)</title>recall</a> : <a xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="417.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.quality.fScore"><title>Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall)</title>fScore</a> : <a xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text></g><g><rect filter="url(#shadowB)" rx="4" ry="4" x="596.4" width="110.00000000000001" y="350.0" style="fill:#f0f8ff;stroke:black;stroke-width:1" height="92.0"/><line y1="378.0" x1="596.4" y2="378.0" x2="706.4" style="stroke:dimgrey;stroke-width:1"/><text x="651.4" y="370.0" fill="black" class="diagram-class-title">Repository</text><text x="602.4" y="392.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.repository.url"><title>URI of an external repository which contains further details about the genetics data</title>url</a> : <a xlink:href="datatypes.html#uri">uri</a> [0..1]</text><text x="602.4" y="406.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.repository.name"><title>URI of an external repository which contains further details about the genetics data</title>name</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="602.4" y="420.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.repository.variantId"><title>Id of the variant in this external repository</title>variantId</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="602.4" y="434.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.repository.readId"><title>Id of the read in this external repository</title>readId</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text></g><g><rect filter="url(#shadowB)" rx="4" ry="4" x="307.0" width="162.8" y="484.0" style="fill:#f0f8ff;stroke:black;stroke-width:1" height="78.0"/><line y1="512.0" x1="307.0" y2="512.0" x2="469.8" style="stroke:dimgrey;stroke-width:1"/><text x="388.4" y="504.0" fill="black" class="diagram-class-title">StructureVariant</text><text x="313.0" y="526.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.structureVariant.precisionOfBoundaries"><title>Precision of boundaries</title>precisionOfBoundaries</a> : <a xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="313.0" y="540.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio"><title>Structural Variant reported aCGH ratio</title>reportedaCGHRatio</a> : <a xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="313.0" y="554.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.structureVariant.length"><title>Structural Variant Length</title>length</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text><g><rect filter="url(#shadowB)" rx="4" ry="4" x="90.19999999999996" width="96.80000000000001" y="491.0" style="fill:#f0f8ff;stroke:black;stroke-width:1" height="64.0"/><line y1="519.0" x1="90.19999999999996" y2="519.0" x2="186.99999999999997" style="stroke:dimgrey;stroke-width:1"/><text x="138.59999999999997" y="511.0" fill="black" class="diagram-class-title">Outer</text><text x="96.19999999999996" y="533.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.structureVariant.outer.start"><title>Structural Variant Outer Start-End</title>start</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="96.19999999999996" y="547.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.structureVariant.outer.end"><title>Structural Variant Outer Start-End</title>end</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text></g><g><rect filter="url(#shadowB)" rx="4" ry="4" x="589.8" width="96.80000000000001" y="491.0" style="fill:#f0f8ff;stroke:black;stroke-width:1" height="64.0"/><line y1="519.0" x1="589.8" y2="519.0" x2="686.5999999999999" style="stroke:dimgrey;stroke-width:1"/><text x="638.1999999999999" y="511.0" fill="black" class="diagram-class-title">Inner</text><text x="595.8" y="533.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.structureVariant.inner.start"><title>Structural Variant Inner Start-End</title>start</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="595.8" y="547.0" fill="black" class="diagram-class-detail"><a xlink:href="sequence-definitions.html#Sequence.structureVariant.inner.end"><title>Structural Variant Inner Start-End</title>end</a> : <a xlink:href="datatypes.html#integer">integer</a> [0..1]</text></g></g></g><line y1="228.0" x1="388.4" y2="147.99999999999997" x2="388.4" style="stroke:navy;stroke-width:1"/><polygon transform="rotate(-90.0 388.4 228.0)" style="fill:navy;stroke:navy;stroke-width:1" points="388.4,228.0 394.4,232.0 400.4,228.0 394.4,224.0 388.4,228.0"/><rect x="362.0" width="52.0" y="181.0" style="fill:white;stroke:black;stroke-width:0" height="18.0"/><text x="388.0" y="191.0" fill="black" class="diagram-class-linkage"><a xlink:href="sequence-definitions.html#Sequence.referenceSeq"><title>A reference sequence is a sequence that is used to represent an allele or variant</title>referenceSeq</a></text><text x="388.4" y="161.99999999999997" fill="black" class="diagram-class-linkage">[0..1]</text><line y1="306.5157813702848" x1="476.4" y2="299.6181678214011" x2="540.4" style="stroke:navy;stroke-width:1"/><polygon transform="rotate(-6.151321210260629 476.4 306.5157813702848)" style="fill:navy;stroke:navy;stroke-width:1" points="476.4,306.5157813702848 482.4,310.5157813702848 488.4,306.5157813702848 482.4,302.5157813702848 476.4,306.5157813702848"/><rect x="493.0" width="30.0" y="296.0" style="fill:white;stroke:black;stroke-width:0" height="18.0"/><text x="508.0" y="306.0" fill="black" class="diagram-class-linkage"><a xlink:href="sequence-definitions.html#Sequence.variant"><title>A' is a variant (mutation) of A = definition every instance of A' is either an immediate mutation of some instance of A, or there is a chain of immediate mutation processes linking A' to some instance of A ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of))</title>variant</a></text><text x="520.4" y="295.6181678214011" fill="black" class="diagram-class-linkage">[0..*]</text><line y1="316.0" x1="300.4" y2="316.0" x2="180.4" style="stroke:navy;stroke-width:1"/><polygon transform="rotate(180.0 300.4 316.0)" style="fill:navy;stroke:navy;stroke-width:1" points="300.4,316.0 306.4,320.0 312.4,316.0 306.4,312.0 300.4,316.0"/><rect x="225.0" width="30.0" y="309.0" style="fill:white;stroke:black;stroke-width:0" height="18.0"/><text x="240.0" y="319.0" fill="black" class="diagram-class-linkage"><a xlink:href="sequence-definitions.html#Sequence.quality"><title>An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686))</title>quality</a></text><text x="195.4" y="312.0" fill="black" class="diagram-class-linkage">[0..*]</text><line y1="342.7680608365019" x1="476.4" y2="379.2699619771863" x2="596.4" style="stroke:navy;stroke-width:1"/><polygon transform="rotate(16.918843707349374 476.4 342.7680608365019)" style="fill:navy;stroke:navy;stroke-width:1" points="476.4,342.7680608365019 482.4,346.7680608365019 488.4,342.7680608365019 482.4,338.7680608365019 476.4,342.7680608365019"/><rect x="514.0" width="44.0" y="354.0" style="fill:white;stroke:black;stroke-width:0" height="18.0"/><text x="536.0" y="364.0" fill="black" class="diagram-class-linkage"><a xlink:href="sequence-definitions.html#Sequence.repository"><title>Configurations of the external repository</title>repository</a></text><text x="576.4" y="375.2699619771863" fill="black" class="diagram-class-linkage">[0..*]</text><line y1="523.0" x1="307.0" y2="523.0" x2="186.99999999999997" style="stroke:navy;stroke-width:1"/><polygon transform="rotate(180.0 307.0 523.0)" style="fill:navy;stroke:navy;stroke-width:1" points="307.0,523.0 313.0,527.0 319.0,523.0 313.0,519.0 307.0,523.0"/><rect x="236.0" width="22.0" y="516.0" style="fill:white;stroke:black;stroke-width:0" height="18.0"/><text x="247.0" y="526.0" fill="black" class="diagram-class-linkage"><a xlink:href="sequence-definitions.html#Sequence.structureVariant.outer"><title>Structural variant outer</title>outer</a></text><text x="201.99999999999997" y="519.0" fill="black" class="diagram-class-linkage">[0..1]</text><line y1="523.0000000000001" x1="469.8" y2="523.0000000000001" x2="589.8" style="stroke:navy;stroke-width:1"/><polygon transform="rotate(0.0 469.8 523.0000000000001)" style="fill:navy;stroke:navy;stroke-width:1" points="469.8,523.0000000000001 475.8,527.0000000000001 481.8,523.0000000000001 475.8,519.0000000000001 469.8,523.0000000000001"/><rect x="518.0" width="22.0" y="516.0" style="fill:white;stroke:black;stroke-width:0" height="18.0"/><text x="529.0" y="526.0" fill="black" class="diagram-class-linkage"><a xlink:href="sequence-definitions.html#Sequence.structureVariant.inner"><title>Structural variant inner</title>inner</a></text><text x="569.8" y="519.0000000000001" fill="black" class="diagram-class-linkage">[0..1]</text><line y1="403.99999999999994" x1="388.4" y2="484.00000000000006" x2="388.4" style="stroke:navy;stroke-width:1"/><polygon transform="rotate(90.0 388.4 403.99999999999994)" style="fill:navy;stroke:navy;stroke-width:1" points="388.4,403.99999999999994 394.4,407.99999999999994 400.4,403.99999999999994 394.4,399.99999999999994 388.4,403.99999999999994"/><rect x="353.0" width="70.0" y="437.0" style="fill:white;stroke:black;stroke-width:0" height="18.0"/><text x="388.0" y="447.0" fill="black" class="diagram-class-linkage"><a xlink:href="sequence-definitions.html#Sequence.structureVariant"><title>Structural variant</title>structureVariant</a></text><text x="388.4" y="480.00000000000006" fill="black" class="diagram-class-linkage">[0..*]</text></svg>
 </div>
</div>

<div id="xml">
<a name="xml"> </a>
<p><b>XML Template</b></p>
 <div id="xml-inner">
  <pre class="spec">
&lt;<a href="sequence-definitions.html#Sequence" title="Raw data describing a biological sequence." class="dict"><b>Sequence</b></a> xmlns=&quot;http://hl7.org/fhir&quot;&gt; <span style="float: right"><a href="xml.html" title="Documentation for this format"><img src="help.png" alt="doco"/></a></span>
 &lt;!-- from <a href="resource.html">Resource</a>: <a href="resource.html#id">id</a>, <a href="resource.html#meta">meta</a>, <a href="resource.html#implicitRules">implicitRules</a>, and <a href="resource.html#language">language</a> --&gt;
 &lt;!-- from <a href="domainresource.html">DomainResource</a>: <a href="narrative.html#Narrative">text</a>, <a href="references.html#contained">contained</a>, <a href="extensibility.html">extension</a>, and <a href="extensibility.html#modifierExtension">modifierExtension</a> --&gt;
 &lt;<a href="sequence-definitions.html#Sequence.identifier" title="A unique identifier for this particular sequence instance." class="dict"><b>identifier</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: darkgreen"><a href="datatypes.html#Identifier">Identifier</a></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Unique ID for this particular sequence</a></span><span style="color: Gray"> --&gt;</span>&lt;/identifier&gt;
 &lt;<a href="sequence-definitions.html#Sequence.type" title="Amino acid / cDNA transcript / RNA variant." class="dict"><b>type</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="valueset-sequence-type.html">AA | DNA | RNA</a></span><span style="color: Gray"> --&gt;</span>
 &lt;<a href="sequence-definitions.html#Sequence.coordinateSystem" title="Whether the sequence is numbered starting at 0 (0-based numbering or coordinates) or starting at 1 (1-based numbering). Values are &quot;0&quot; for 0-based numbering and &quot;1&quot; for one-based." class="dict"><b>coordinateSystem</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Numbering used for sequence (0-based or 1-based)</a></span><span style="color: Gray"> --&gt;</span>
 &lt;<a href="sequence-definitions.html#Sequence.patient" title="The patient whose sequencing results are described by this resource." class="dict"><b>patient</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="patient.html#Patient">Patient</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Who and/or what this is about</a></span><span style="color: Gray"> --&gt;</span>&lt;/patient&gt;
 &lt;<a href="sequence-definitions.html#Sequence.specimen" title="Specimen used for sequencing." class="dict"><b>specimen</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="specimen.html#Specimen">Specimen</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Specimen used for sequencing</a></span><span style="color: Gray"> --&gt;</span>&lt;/specimen&gt;
 &lt;<a href="sequence-definitions.html#Sequence.device" title="The method for sequencing, for example, chip information." class="dict"><b>device</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="device.html#Device">Device</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">The method for sequencing</a></span><span style="color: Gray"> --&gt;</span>&lt;/device&gt;
 &lt;<a href="sequence-definitions.html#Sequence.quantity" title="Quantity of the sequence." class="dict"><b>quantity</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#Quantity">Quantity</a></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Quantity of the sequence</a></span><span style="color: Gray"> --&gt;</span>&lt;/quantity&gt;
 &lt;<a href="sequence-definitions.html#Sequence.referenceSeq" title="A reference sequence is a sequence that is used to represent an allele or variant." class="dict"><b>referenceSeq</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..1</b></span> Reference sequence --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.chromosome" title="Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340))." class="dict"><b>chromosome</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> <span style="color: navy"><a style="color: navy" href="valueset-chromosome-human.html">Chromosome containing genetic finding</a></span><span style="color: Gray"> --&gt;</span>&lt;/chromosome&gt;
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild" title="The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used." class="dict"><b>genomeBuild</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId" title="Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences." class="dict"><b>referenceSeqId</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> <span style="color: navy"><a style="color: navy" href="valueset-sequence-referenceSeq.html">Reference identifier</a></span><span style="color: Gray"> --&gt;</span>&lt;/referenceSeqId&gt;
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer" title="A Pointer to another Sequence entity as refence sequence." class="dict"><b>referenceSeqPointer</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">A Pointer to another Sequence entity as refence sequence</a></span><span style="color: Gray"> --&gt;</span>&lt;/referenceSeqPointer&gt;
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString" title="A Reference Sequence string." class="dict"><b>referenceSeqString</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">A Reference Sequence string</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.strand" title="Strand of DNA. Available values are &quot;1&quot; for the plus strand and &quot;-1&quot; for the minus strand." class="dict"><b>strand</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Strand of DNA</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.windowStart" title="Start position (inclusive) of the window on the reference sequence." class="dict"><b>windowStart</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Start position (inclusive) of the window on the  reference sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.windowEnd" title="End position (exclusive) of the window on the reference sequence." class="dict"><b>windowEnd</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">End position (exclusive) of the window on the reference sequence</a></span><span style="color: Gray"> --&gt;</span>
 &lt;/referenceSeq&gt;
 &lt;<a href="sequence-definitions.html#Sequence.variant" title="A' is a variant (mutation) of A = definition every instance of A' is either an immediate mutation of some instance of A, or there is a chain of immediate mutation processes linking A' to some instance of A ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of))." class="dict"><b>variant</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..*</b></span> Sequence variant --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.start" title="Start position (inclusive) of the variant on the  reference sequence." class="dict"><b>start</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Start position (inclusive) of the variant on the  reference sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.end" title="End position (exclusive) of the variant on the reference sequence." class="dict"><b>end</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">End position (exclusive) of the variant on the reference sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.observedAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand." class="dict"><b>observedAllele</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Allele that was observed</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.referenceAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand." class="dict"><b>referenceAllele</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Allele of reference sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.cigar" title="Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm)." class="dict"><b>cigar</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Extended CIGAR string for aligning the sequence with reference bases</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.variantPointer" title="A pointer to an Observation containing variant information." class="dict"><b>variantPointer</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="observation.html#Observation">Observation</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Pointer to observed variant information</a></span><span style="color: Gray"> --&gt;</span>&lt;/variantPointer&gt;
 &lt;/variant&gt;
 &lt;<a href="sequence-definitions.html#Sequence.observedSeq" title="Sequence that was observed." class="dict"><b>observedSeq</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Observed sequence</a></span><span style="color: Gray"> --&gt;</span>
 &lt;<a href="sequence-definitions.html#Sequence.quality" title="An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686))." class="dict"><b>quality</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..*</b></span> Sequence quality --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.standardSequence" title="Gold standard sequence used for comparing against." class="dict"><b>standardSequence</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Standard sequence for comparison</a></span><span style="color: Gray"> --&gt;</span>&lt;/standardSequence&gt;
  &lt;<a href="sequence-definitions.html#Sequence.quality.start" title="Start position (inclusive) of the sequence." class="dict"><b>start</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Start position (inclusive) of the sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.end" title="End position (exclusive) of the sequence." class="dict"><b>end</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">End position (exclusive) of the sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.score" title="The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685))." class="dict"><b>score</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#Quantity">Quantity</a></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Quality score</a></span><span style="color: Gray"> --&gt;</span>&lt;/score&gt;
  &lt;<a href="sequence-definitions.html#Sequence.quality.method" title="Method for quality." class="dict"><b>method</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Method for quality</a></span><span style="color: Gray"> --&gt;</span>&lt;/method&gt;
  &lt;<a href="sequence-definitions.html#Sequence.quality.truthTP" title="True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict"><b>truthTP</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">True positives from the perspective of the truth data</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.queryTP" title="True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict"><b>queryTP</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">True positives from the perspective of the query data</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.truthFN" title="False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here." class="dict"><b>truthFN</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">False negatives</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.queryFP" title="False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here." class="dict"><b>queryFP</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">False positives</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.gtFP" title="The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar)." class="dict"><b>gtFP</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">False positives where the non-REF alleles in the Truth and Query Call Sets match</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.precision" title="QUERY.TP / (QUERY.TP + QUERY.FP)." class="dict"><b>precision</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Precision</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.recall" title="TRUTH.TP / (TRUTH.TP + TRUTH.FN)." class="dict"><b>recall</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Recall</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.fScore" title="Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall)." class="dict"><b>fScore</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">F-score</a></span><span style="color: Gray"> --&gt;</span>
 &lt;/quality&gt;
 &lt;<a href="sequence-definitions.html#Sequence.readCoverage" title="Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence." class="dict"><b>readCoverage</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Average number of reads representing a given nucleotide in the reconstructed sequence</a></span><span style="color: Gray"> --&gt;</span>
 &lt;<a href="sequence-definitions.html#Sequence.repository" title="Configurations of the external repository." class="dict"><b>repository</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..*</b></span> External repository --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.url" title="URI of an external repository which contains further details about the genetics data." class="dict"><b>url</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#uri">uri</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">URI of the repository</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.name" title="URI of an external repository which contains further details about the genetics data." class="dict"><b>name</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Name of the repository</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.variantId" title="Id of the variant in this external repository." class="dict"><b>variantId</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Id of the variant</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.readId" title="Id of the read in this external repository." class="dict"><b>readId</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Id of the read</a></span><span style="color: Gray"> --&gt;</span>
 &lt;/repository&gt;
 &lt;<a href="sequence-definitions.html#Sequence.pointer" title="Pointer to next atomic sequence which at most contains one variant." class="dict"><b>pointer</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Pointer to next atomic sequence</a></span><span style="color: Gray"> --&gt;</span>&lt;/pointer&gt;
 &lt;<a href="sequence-definitions.html#Sequence.structureVariant" title="Structural variant." class="dict"><b>structureVariant</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..*</b></span> Structural variant --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.precisionOfBoundaries" title="Precision of boundaries." class="dict"><b>precisionOfBoundaries</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Precision of boundaries</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio" title="Structural Variant reported aCGH ratio." class="dict"><b>reportedaCGHRatio</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant reported aCGH ratio</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.length" title="Structural Variant Length." class="dict"><b>length</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Length</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.outer" title="Structural variant outer." class="dict"><b>outer</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..1</b></span> Structural variant outer --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.structureVariant.outer.start" title="Structural Variant Outer Start-End." class="dict"><b>start</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Outer Start-End</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.structureVariant.outer.end" title="Structural Variant Outer Start-End." class="dict"><b>end</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Outer Start-End</a></span><span style="color: Gray"> --&gt;</span>
  &lt;/outer&gt;
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.inner" title="Structural variant inner." class="dict"><b>inner</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..1</b></span> Structural variant inner --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.structureVariant.inner.start" title="Structural Variant Inner Start-End." class="dict"><b>start</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Inner Start-End</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.structureVariant.inner.end" title="Structural Variant Inner Start-End." class="dict"><b>end</b></a> value=&quot;[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]&quot;/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Inner Start-End</a></span><span style="color: Gray"> --&gt;</span>
  &lt;/inner&gt;
 &lt;/structureVariant&gt;
&lt;/Sequence&gt;
</pre>

 </div>
</div>

<div id="json2">
<a name="json"> </a>
<p><b>JSON Template</b></p>
 <div id="json-inner">
  <pre class="spec">
{<span style="float: right"><a href="json.html" title="Documentation for this format"><img src="help.png" alt="doco"/></a></span>
  &quot;resourceType&quot; : &quot;<a href="sequence-definitions.html#Sequence" title="Raw data describing a biological sequence." class="dict"><b>Sequence</b></a>&quot;,
  // from <a href="resource.html">Resource</a>: <a href="resource.html#id">id</a>, <a href="resource.html#meta">meta</a>, <a href="resource.html#implicitRules">implicitRules</a>, and <a href="resource.html#language">language</a>
  // from <a href="domainresource.html">DomainResource</a>: <a href="narrative.html#Narrative">text</a>, <a href="references.html#contained">contained</a>, <a href="extensibility.html">extension</a>, and <a href="extensibility.html#modifierExtension">modifierExtension</a>
  &quot;<a href="sequence-definitions.html#Sequence.identifier" title="A unique identifier for this particular sequence instance." class="dict">identifier</a>&quot; : [{ <span style="color: darkgreen"><a href="datatypes.html#Identifier">Identifier</a></span> }], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Unique ID for this particular sequence</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.type" title="Amino acid / cDNA transcript / RNA variant." class="dict">type</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-sequence-type.html">AA | DNA | RNA</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.coordinateSystem" title="Whether the sequence is numbered starting at 0 (0-based numbering or coordinates) or starting at 1 (1-based numbering). Values are &quot;0&quot; for 0-based numbering and &quot;1&quot; for one-based." class="dict">coordinateSystem</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Numbering used for sequence (0-based or 1-based)</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.patient" title="The patient whose sequencing results are described by this resource." class="dict">patient</a>&quot; : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="patient.html#Patient">Patient</a>)</span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Who and/or what this is about</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.specimen" title="Specimen used for sequencing." class="dict">specimen</a>&quot; : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="specimen.html#Specimen">Specimen</a>)</span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Specimen used for sequencing</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.device" title="The method for sequencing, for example, chip information." class="dict">device</a>&quot; : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="device.html#Device">Device</a>)</span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">The method for sequencing</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.quantity" title="Quantity of the sequence." class="dict">quantity</a>&quot; : { <span style="color: darkgreen"><a href="datatypes.html#Quantity">Quantity</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Quantity of the sequence</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.referenceSeq" title="A reference sequence is a sequence that is used to represent an allele or variant." class="dict">referenceSeq</a>&quot; : { <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Reference sequence</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.referenceSeq.chromosome" title="Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340))." class="dict">chromosome</a>&quot; : { <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-chromosome-human.html">Chromosome containing genetic finding</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild" title="The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used." class="dict">genomeBuild</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId" title="Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences." class="dict">referenceSeqId</a>&quot; : { <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> }, <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-sequence-referenceSeq.html">Reference identifier</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer" title="A Pointer to another Sequence entity as refence sequence." class="dict">referenceSeqPointer</a>&quot; : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>)</span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">A Pointer to another Sequence entity as refence sequence</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString" title="A Reference Sequence string." class="dict">referenceSeqString</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">A Reference Sequence string</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.referenceSeq.strand" title="Strand of DNA. Available values are &quot;1&quot; for the plus strand and &quot;-1&quot; for the minus strand." class="dict">strand</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Strand of DNA</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.referenceSeq.windowStart" title="Start position (inclusive) of the window on the reference sequence." class="dict">windowStart</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Start position (inclusive) of the window on the  reference sequence</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.referenceSeq.windowEnd" title="End position (exclusive) of the window on the reference sequence." class="dict">windowEnd</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt; <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">End position (exclusive) of the window on the reference sequence</a></span>
  },
  &quot;<a href="sequence-definitions.html#Sequence.variant" title="A' is a variant (mutation) of A = definition every instance of A' is either an immediate mutation of some instance of A, or there is a chain of immediate mutation processes linking A' to some instance of A ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of))." class="dict">variant</a>&quot; : [{ <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Sequence variant</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.variant.start" title="Start position (inclusive) of the variant on the  reference sequence." class="dict">start</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Start position (inclusive) of the variant on the  reference sequence</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.variant.end" title="End position (exclusive) of the variant on the reference sequence." class="dict">end</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">End position (exclusive) of the variant on the reference sequence</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.variant.observedAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand." class="dict">observedAllele</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Allele that was observed</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.variant.referenceAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand." class="dict">referenceAllele</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Allele of reference sequence</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.variant.cigar" title="Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm)." class="dict">cigar</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Extended CIGAR string for aligning the sequence with reference bases</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.variant.variantPointer" title="A pointer to an Observation containing variant information." class="dict">variantPointer</a>&quot; : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="observation.html#Observation">Observation</a>)</span> } <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Pointer to observed variant information</a></span>
  }],
  &quot;<a href="sequence-definitions.html#Sequence.observedSeq" title="Sequence that was observed." class="dict">observedSeq</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Observed sequence</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.quality" title="An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686))." class="dict">quality</a>&quot; : [{ <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Sequence quality</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.standardSequence" title="Gold standard sequence used for comparing against." class="dict">standardSequence</a>&quot; : { <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Standard sequence for comparison</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.start" title="Start position (inclusive) of the sequence." class="dict">start</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Start position (inclusive) of the sequence</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.end" title="End position (exclusive) of the sequence." class="dict">end</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">End position (exclusive) of the sequence</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.score" title="The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685))." class="dict">score</a>&quot; : { <span style="color: darkgreen"><a href="datatypes.html#Quantity">Quantity</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Quality score</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.method" title="Method for quality." class="dict">method</a>&quot; : { <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Method for quality</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.truthTP" title="True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict">truthTP</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">True positives from the perspective of the truth data</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.queryTP" title="True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict">queryTP</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">True positives from the perspective of the query data</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.truthFN" title="False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here." class="dict">truthFN</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">False negatives</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.queryFP" title="False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here." class="dict">queryFP</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">False positives</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.gtFP" title="The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar)." class="dict">gtFP</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">False positives where the non-REF alleles in the Truth and Query Call Sets match</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.precision" title="QUERY.TP / (QUERY.TP + QUERY.FP)." class="dict">precision</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Precision</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.recall" title="TRUTH.TP / (TRUTH.TP + TRUTH.FN)." class="dict">recall</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Recall</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.quality.fScore" title="Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall)." class="dict">fScore</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt; <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">F-score</a></span>
  }],
  &quot;<a href="sequence-definitions.html#Sequence.readCoverage" title="Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence." class="dict">readCoverage</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Average number of reads representing a given nucleotide in the reconstructed sequence</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.repository" title="Configurations of the external repository." class="dict">repository</a>&quot; : [{ <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">External repository</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.repository.url" title="URI of an external repository which contains further details about the genetics data." class="dict">url</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#uri">uri</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">URI of the repository</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.repository.name" title="URI of an external repository which contains further details about the genetics data." class="dict">name</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Name of the repository</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.repository.variantId" title="Id of the variant in this external repository." class="dict">variantId</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Id of the variant</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.repository.readId" title="Id of the read in this external repository." class="dict">readId</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot; <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Id of the read</a></span>
  }],
  &quot;<a href="sequence-definitions.html#Sequence.pointer" title="Pointer to next atomic sequence which at most contains one variant." class="dict">pointer</a>&quot; : [{ <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>)</span> }], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Pointer to next atomic sequence</a></span>
  &quot;<a href="sequence-definitions.html#Sequence.structureVariant" title="Structural variant." class="dict">structureVariant</a>&quot; : [{ <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural variant</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.structureVariant.precisionOfBoundaries" title="Precision of boundaries." class="dict">precisionOfBoundaries</a>&quot; : &quot;&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;&quot;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Precision of boundaries</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio" title="Structural Variant reported aCGH ratio." class="dict">reportedaCGHRatio</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant reported aCGH ratio</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.structureVariant.length" title="Structural Variant Length." class="dict">length</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Length</a></span>
    &quot;<a href="sequence-definitions.html#Sequence.structureVariant.outer" title="Structural variant outer." class="dict">outer</a>&quot; : { <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural variant outer</a></span>
      &quot;<a href="sequence-definitions.html#Sequence.structureVariant.outer.start" title="Structural Variant Outer Start-End." class="dict">start</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Outer Start-End</a></span>
      &quot;<a href="sequence-definitions.html#Sequence.structureVariant.outer.end" title="Structural Variant Outer Start-End." class="dict">end</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt; <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Outer Start-End</a></span>
    },
    &quot;<a href="sequence-definitions.html#Sequence.structureVariant.inner" title="Structural variant inner." class="dict">inner</a>&quot; : { <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural variant inner</a></span>
      &quot;<a href="sequence-definitions.html#Sequence.structureVariant.inner.start" title="Structural Variant Inner Start-End." class="dict">start</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Inner Start-End</a></span>
      &quot;<a href="sequence-definitions.html#Sequence.structureVariant.inner.end" title="Structural Variant Inner Start-End." class="dict">end</a>&quot; : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt; <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Inner Start-End</a></span>
    }
  }]
}
</pre>

 </div>
</div>
  
<div id="ttl2">
<a name="ttl"> </a>
<p><b>Turtle Template</b></p>
 <div id="ttl-inner">
  <pre class="spec">
@prefix fhir: &lt;http://hl7.org/fhir/&gt; .<span style="float: right"><a href="rdf.html" title="Documentation for this format"><img src="help.png" alt="doco"/></a></span>


[ a fhir:<a href="sequence-definitions.html#Sequence" title="Raw data describing a biological sequence." class="dict"><b>Sequence</b></a>;
  fhir:nodeRole fhir:treeRoot; # if this is the parser root

  # from <a href="resource.html">Resource</a>: <a href="resource.html#id">.id</a>, <a href="resource.html#meta">.meta</a>, <a href="resource.html#implicitRules">.implicitRules</a>, and <a href="resource.html#language">.language</a>
  # from <a href="domainresource.html">DomainResource</a>: <a href="narrative.html#Narrative">.text</a>, <a href="references.html#contained">.contained</a>, <a href="extensibility.html">.extension</a>, and <a href="extensibility.html#modifierExtension">.modifierExtension</a>
  fhir:<a href="sequence-definitions.html#Sequence.identifier" title="A unique identifier for this particular sequence instance." class="dict">Sequence.identifier</a><a name="ttl-identifier"> </a>[ <a href="datatypes.html#Identifier">Identifier</a> ], ... ; # 0..* <span style="color: navy">Unique ID for this particular sequence</span>
  fhir:<a href="sequence-definitions.html#Sequence.type" title="Amino acid / cDNA transcript / RNA variant." class="dict">Sequence.type</a><a name="ttl-type"> </a>[ <a href="datatypes.html#code">code</a> ]; # 1..1 <span style="color: navy">AA | DNA | RNA</span>
  fhir:<a href="sequence-definitions.html#Sequence.coordinateSystem" title="Whether the sequence is numbered starting at 0 (0-based numbering or coordinates) or starting at 1 (1-based numbering). Values are &quot;0&quot; for 0-based numbering and &quot;1&quot; for one-based." class="dict">Sequence.coordinateSystem</a><a name="ttl-coordinateSystem"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 1..1 <span style="color: navy">Numbering used for sequence (0-based or 1-based)</span>
  fhir:<a href="sequence-definitions.html#Sequence.patient" title="The patient whose sequencing results are described by this resource." class="dict">Sequence.patient</a><a name="ttl-patient"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="patient.html#Patient">Patient</a>) ]; # 0..1 <span style="color: navy">Who and/or what this is about</span>
  fhir:<a href="sequence-definitions.html#Sequence.specimen" title="Specimen used for sequencing." class="dict">Sequence.specimen</a><a name="ttl-specimen"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="specimen.html#Specimen">Specimen</a>) ]; # 0..1 <span style="color: navy">Specimen used for sequencing</span>
  fhir:<a href="sequence-definitions.html#Sequence.device" title="The method for sequencing, for example, chip information." class="dict">Sequence.device</a><a name="ttl-device"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="device.html#Device">Device</a>) ]; # 0..1 <span style="color: navy">The method for sequencing</span>
  fhir:<a href="sequence-definitions.html#Sequence.quantity" title="Quantity of the sequence." class="dict">Sequence.quantity</a><a name="ttl-quantity"> </a>[ <a href="datatypes.html#Quantity">Quantity</a> ]; # 0..1 <span style="color: navy">Quantity of the sequence</span>
  fhir:<a href="sequence-definitions.html#Sequence.referenceSeq" title="A reference sequence is a sequence that is used to represent an allele or variant." class="dict">Sequence.referenceSeq</a><a name="ttl-referenceSeq"> </a>[ # 0..1 <span style="color: navy">Reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.chromosome" title="Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340))." class="dict">Sequence.referenceSeq.chromosome</a><a name="ttl-chromosome"> </a>[ <a href="datatypes.html#CodeableConcept">CodeableConcept</a> ]; # 0..1 <span style="color: navy">Chromosome containing genetic finding</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild" title="The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used." class="dict">Sequence.referenceSeq.genomeBuild</a><a name="ttl-genomeBuild"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId" title="Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences." class="dict">Sequence.referenceSeq.referenceSeqId</a><a name="ttl-referenceSeqId"> </a>[ <a href="datatypes.html#CodeableConcept">CodeableConcept</a> ]; # 1..1 <span style="color: navy">Reference identifier</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer" title="A Pointer to another Sequence entity as refence sequence." class="dict">Sequence.referenceSeq.referenceSeqPointer</a><a name="ttl-referenceSeqPointer"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>) ]; # 0..1 <span style="color: navy">A Pointer to another Sequence entity as refence sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString" title="A Reference Sequence string." class="dict">Sequence.referenceSeq.referenceSeqString</a><a name="ttl-referenceSeqString"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">A Reference Sequence string</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.strand" title="Strand of DNA. Available values are &quot;1&quot; for the plus strand and &quot;-1&quot; for the minus strand." class="dict">Sequence.referenceSeq.strand</a><a name="ttl-strand"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 1..1 <span style="color: navy">Strand of DNA</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.windowStart" title="Start position (inclusive) of the window on the reference sequence." class="dict">Sequence.referenceSeq.windowStart</a><a name="ttl-windowStart"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 1..1 <span style="color: navy">Start position (inclusive) of the window on the  reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.windowEnd" title="End position (exclusive) of the window on the reference sequence." class="dict">Sequence.referenceSeq.windowEnd</a><a name="ttl-windowEnd"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 1..1 <span style="color: navy">End position (exclusive) of the window on the reference sequence</span>
  ];
  fhir:<a href="sequence-definitions.html#Sequence.variant" title="A' is a variant (mutation) of A = definition every instance of A' is either an immediate mutation of some instance of A, or there is a chain of immediate mutation processes linking A' to some instance of A ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of))." class="dict">Sequence.variant</a><a name="ttl-variant"> </a>[ # 0..* <span style="color: navy">Sequence variant</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.start" title="Start position (inclusive) of the variant on the  reference sequence." class="dict">Sequence.variant.start</a><a name="ttl-start"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Start position (inclusive) of the variant on the  reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.end" title="End position (exclusive) of the variant on the reference sequence." class="dict">Sequence.variant.end</a><a name="ttl-end"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">End position (exclusive) of the variant on the reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.observedAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand." class="dict">Sequence.variant.observedAllele</a><a name="ttl-observedAllele"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Allele that was observed</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.referenceAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand." class="dict">Sequence.variant.referenceAllele</a><a name="ttl-referenceAllele"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Allele of reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.cigar" title="Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm)." class="dict">Sequence.variant.cigar</a><a name="ttl-cigar"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Extended CIGAR string for aligning the sequence with reference bases</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.variantPointer" title="A pointer to an Observation containing variant information." class="dict">Sequence.variant.variantPointer</a><a name="ttl-variantPointer"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="observation.html#Observation">Observation</a>) ]; # 0..1 <span style="color: navy">Pointer to observed variant information</span>
  ], ...;
  fhir:<a href="sequence-definitions.html#Sequence.observedSeq" title="Sequence that was observed." class="dict">Sequence.observedSeq</a><a name="ttl-observedSeq"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Observed sequence</span>
  fhir:<a href="sequence-definitions.html#Sequence.quality" title="An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686))." class="dict">Sequence.quality</a><a name="ttl-quality"> </a>[ # 0..* <span style="color: navy">Sequence quality</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.standardSequence" title="Gold standard sequence used for comparing against." class="dict">Sequence.quality.standardSequence</a><a name="ttl-standardSequence"> </a>[ <a href="datatypes.html#CodeableConcept">CodeableConcept</a> ]; # 0..1 <span style="color: navy">Standard sequence for comparison</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.start" title="Start position (inclusive) of the sequence." class="dict">Sequence.quality.start</a><a name="ttl-start"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Start position (inclusive) of the sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.end" title="End position (exclusive) of the sequence." class="dict">Sequence.quality.end</a><a name="ttl-end"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">End position (exclusive) of the sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.score" title="The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685))." class="dict">Sequence.quality.score</a><a name="ttl-score"> </a>[ <a href="datatypes.html#Quantity">Quantity</a> ]; # 0..1 <span style="color: navy">Quality score</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.method" title="Method for quality." class="dict">Sequence.quality.method</a><a name="ttl-method"> </a>[ <a href="datatypes.html#CodeableConcept">CodeableConcept</a> ]; # 0..1 <span style="color: navy">Method for quality</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.truthTP" title="True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict">Sequence.quality.truthTP</a><a name="ttl-truthTP"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">True positives from the perspective of the truth data</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.queryTP" title="True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict">Sequence.quality.queryTP</a><a name="ttl-queryTP"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">True positives from the perspective of the query data</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.truthFN" title="False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here." class="dict">Sequence.quality.truthFN</a><a name="ttl-truthFN"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">False negatives</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.queryFP" title="False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here." class="dict">Sequence.quality.queryFP</a><a name="ttl-queryFP"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">False positives</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.gtFP" title="The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar)." class="dict">Sequence.quality.gtFP</a><a name="ttl-gtFP"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">False positives where the non-REF alleles in the Truth and Query Call Sets match</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.precision" title="QUERY.TP / (QUERY.TP + QUERY.FP)." class="dict">Sequence.quality.precision</a><a name="ttl-precision"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">Precision</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.recall" title="TRUTH.TP / (TRUTH.TP + TRUTH.FN)." class="dict">Sequence.quality.recall</a><a name="ttl-recall"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">Recall</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.fScore" title="Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall)." class="dict">Sequence.quality.fScore</a><a name="ttl-fScore"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">F-score</span>
  ], ...;
  fhir:<a href="sequence-definitions.html#Sequence.readCoverage" title="Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence." class="dict">Sequence.readCoverage</a><a name="ttl-readCoverage"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Average number of reads representing a given nucleotide in the reconstructed sequence</span>
  fhir:<a href="sequence-definitions.html#Sequence.repository" title="Configurations of the external repository." class="dict">Sequence.repository</a><a name="ttl-repository"> </a>[ # 0..* <span style="color: navy">External repository</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.url" title="URI of an external repository which contains further details about the genetics data." class="dict">Sequence.repository.url</a><a name="ttl-url"> </a>[ <a href="datatypes.html#uri">uri</a> ]; # 0..1 <span style="color: navy">URI of the repository</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.name" title="URI of an external repository which contains further details about the genetics data." class="dict">Sequence.repository.name</a><a name="ttl-name"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Name of the repository</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.variantId" title="Id of the variant in this external repository." class="dict">Sequence.repository.variantId</a><a name="ttl-variantId"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Id of the variant</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.readId" title="Id of the read in this external repository." class="dict">Sequence.repository.readId</a><a name="ttl-readId"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Id of the read</span>
  ], ...;
  fhir:<a href="sequence-definitions.html#Sequence.pointer" title="Pointer to next atomic sequence which at most contains one variant." class="dict">Sequence.pointer</a><a name="ttl-pointer"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>) ], ... ; # 0..* <span style="color: navy">Pointer to next atomic sequence</span>
  fhir:<a href="sequence-definitions.html#Sequence.structureVariant" title="Structural variant." class="dict">Sequence.structureVariant</a><a name="ttl-structureVariant"> </a>[ # 0..* <span style="color: navy">Structural variant</span>
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.precisionOfBoundaries" title="Precision of boundaries." class="dict">Sequence.structureVariant.precisionOfBoundaries</a><a name="ttl-precisionOfBoundaries"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Precision of boundaries</span>
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio" title="Structural Variant reported aCGH ratio." class="dict">Sequence.structureVariant.reportedaCGHRatio</a><a name="ttl-reportedaCGHRatio"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">Structural Variant reported aCGH ratio</span>
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.length" title="Structural Variant Length." class="dict">Sequence.structureVariant.length</a><a name="ttl-length"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Length</span>
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.outer" title="Structural variant outer." class="dict">Sequence.structureVariant.outer</a><a name="ttl-outer"> </a>[ # 0..1 <span style="color: navy">Structural variant outer</span>
      fhir:<a href="sequence-definitions.html#Sequence.structureVariant.outer.start" title="Structural Variant Outer Start-End." class="dict">Sequence.structureVariant.outer.start</a><a name="ttl-start"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Outer Start-End</span>
      fhir:<a href="sequence-definitions.html#Sequence.structureVariant.outer.end" title="Structural Variant Outer Start-End." class="dict">Sequence.structureVariant.outer.end</a><a name="ttl-end"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Outer Start-End</span>
    ];
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.inner" title="Structural variant inner." class="dict">Sequence.structureVariant.inner</a><a name="ttl-inner"> </a>[ # 0..1 <span style="color: navy">Structural variant inner</span>
      fhir:<a href="sequence-definitions.html#Sequence.structureVariant.inner.start" title="Structural Variant Inner Start-End." class="dict">Sequence.structureVariant.inner.start</a><a name="ttl-start"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Inner Start-End</span>
      fhir:<a href="sequence-definitions.html#Sequence.structureVariant.inner.end" title="Structural Variant Inner Start-End." class="dict">Sequence.structureVariant.inner.end</a><a name="ttl-end"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Inner Start-End</span>
    ];
  ], ...;
]
</pre>

 </div>
</div>

<div id="vdiff2">
<p><b>Changes since DSTU2</b></p>
 <div id="diff-inner2">
  <p>This resource did not exist in Release 2</p>
 </div>
</div>
  
  </div>
</div>
  
<p>&nbsp;</p>  
<p>Alternate definitions: 
Master Definition (<a href="sequence.profile.xml.html">XML</a>, <a href="sequence.profile.json.html">JSON</a>),
<a href="xml.html">XML</a> <a href="sequence.xsd">Schema</a>/<a href="sequence.sch">Schematron</a> (for ) + <a href="json.html">JSON</a> <a href="Sequence.schema.json">Schema</a>, <a href="sequence.shex.html">ShEx</a> (for <a href="rdf.html">Turtle</a>)</p>


<a name="tx"> </a>
<h3 class="self-link-parent"><span class="sectioncount">10.4.3.1</span> 
Terminology Bindings
 <a href="sequence.html#tx" title="link to here" class="self-link"><img src="assets/images/link.svg" width="20" class="self-link" height="20"/></a></h3>
<table class="grid">
 <tr><th>Path</th><th>Definition</th><th>Type</th><th>Reference</th></tr>
 <tr><td valign="top" title="sequenceType">Sequence.type </td><td valign="top">Type if a sequence -- DNA, RNA, or amino acid sequence</td><td><a href="terminologies.html#example">Example</a></td><td valign="top"><a href="valueset-sequence-type.html">sequenceType</a>  <!-- b --></td> </tr>
 <tr><td valign="top" title="chromosome-human">Sequence.referenceSeq.chromosome </td><td valign="top">Chromosome number for human</td><td><a href="terminologies.html#example">Example</a></td><td valign="top"><a href="valueset-chromosome-human.html">chromosome-human</a>  <!-- b --></td> </tr>
 <tr><td valign="top" title="ENSEMBL">Sequence.referenceSeq.referenceSeqId </td><td valign="top">Reference identifier</td><td><a href="terminologies.html#example">Example</a></td><td valign="top"><a href="valueset-sequence-referenceSeq.html">ENSEMBL</a>  <!-- b --></td> </tr>
</table>
<p> </p>





  <!-- insert xhtml content here -->



<a name="search"> </a>
<h2 class="self-link-parent"><span class="sectioncount">10.4.4</span> Search Parameters <a href="sequence.html#search" title="link to here" class="self-link"><img src="assets/images/link.svg" width="20" class="self-link" height="20"/></a></h2>
<p>Search parameters for this resource. The <a href="search.html#all">common parameters</a> also apply. See <a href="search.html">Searching</a> for more information about searching in REST, messaging, and services.</p>
<table class="list">
<tr><td><b>Name</b></td><td><b>Type</b></td><td><b>Description</b></td><td><b>Paths</b></td></tr>
<tr><td>chromosome</td><td><a href="search.html#token">token</a></td><td>Chromosome of the sequence</td><td>Sequence.referenceSeq.chromosome</td></tr>
<tr><td>coordinate</td><td><a href="search.html#composite">composite</a></td><td>Genomic coordinate of the sequence. For example, a search for sequence in region 1:123-345 can be represented as `coordinate=1$lt345$gt123`</td><td/></tr>
<tr><td>end</td><td><a href="search.html#number">number</a></td><td>End position (0-based exclusive) of the sequence</td><td>Sequence.variant.end</td></tr>
<tr><td>patient</td><td><a href="search.html#reference">reference</a></td><td>The subject that the observation is about</td><td>Sequence.patient<br/>(<a href="patient.html">Patient</a>)</td></tr>
<tr><td>start</td><td><a href="search.html#number">number</a></td><td>Start position (0-based inclusive) of the sequence</td><td>Sequence.variant.start</td></tr>
<tr><td>type</td><td><a href="search.html#token">token</a></td><td>The type of the variant: Amino acid / cDNA transcript / RNA variant.</td><td>Sequence.type</td></tr>
</table>



</div>


				</div>  <!-- /inner-wrapper -->
            </div>  <!-- /row -->
        </div>  <!-- /container -->
        
    </div>  <!-- /segment-content -->


	<div id="segment-footer" class="segment">  <!-- segment-footer -->
		<div class="container">  <!-- container -->
			<div class="inner-wrapper">
				<p>
        &copy; HL7.org 2011+. FHIR STU3 Candidate (v1.7.0-10082) generated on Thu, Oct 27, 2016 16:28+0000. <a href="qa.html">QA Page</a>
        <br/>
        <span style="color: #FFFF77">
        Links: <a style="color: #81BEF7" href="http://hl7.org/fhir/search.cfm">Search <img src="external.png" style="text-align: baseline"/></a> | 
               <a style="color: #81BEF7" href="history.html">Version History</a> | 
               <a style="color: #81BEF7" href="toc.html">Table of Contents</a> | 
               <a style="color: #81BEF7" href="http://services.w3.org/htmldiff?doc1=http%3A%2F%2Fhl7.org%2Ffhir%2FDSTU2%2Fsequence.html&amp;doc2=http%3A%2F%2Fhl7-fhir.github.io%2F%2Fsequence.html">Compare to DSTU2 <img src="external.png" style="text-align: baseline"/></a> |                
               <a rel="license" style="color: #81BEF7" href="license.html"><img src="cc0.png" alt="CC0" style="border-style: none;"/></a> | 
               <a style="color: #81BEF7" href="http://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemAdd&amp;tracker_id=677" target="_blank">Propose a change <img src="external.png" style="text-align: baseline"/></a>   
        </span>
        </p>
			</div>  <!-- /inner-wrapper -->
		</div>  <!-- /container -->
	</div>  <!-- /segment-footer -->
  <!-- disqus thread -->
  <!-- disqus -->
  <!-- end disqus -->        
	
	<div id="segment-post-footer" class="segment hidden">  <!-- segment-post-footer -->
		<div class="container">  <!-- container -->
		</div>  <!-- /container -->
	</div>  <!-- /segment-post-footer -->
    
      <!-- JS and analytics only. -->
      <!-- Bootstrap core JavaScript
================================================== -->
  <!-- Placed at the end of the document so the pages load faster -->
<script src="./assets/js/jquery.js"> </script>     <!-- note keep space here, otherwise it will be transformed to empty tag -> fails -->
<script src="./dist/js/bootstrap.min.js"> </script>
<script src="./assets/js/respond.min.js"> </script>

<script src="./assets/js/fhir.js"> </script>

  <!-- Analytics Below
================================================== -->




<script src="external/jquery/jquery.js"> </script>
<script src="jquery-ui.min.js"> </script>
<script>
try {
  var currentTabIndex = sessionStorage.getItem('fhir-resource-tab-index');
}
catch(exception){ 
}

if (!currentTabIndex)
  currentTabIndex = '0';

$( '#tabs' ).tabs({
         active: currentTabIndex,
         activate: function( event, ui ) {
             var active = $('.selector').tabs('option', 'active');
             currentTabIndex = ui.newTab.index();
             document.activeElement.blur();
             try {
               sessionStorage.setItem('fhir-resource-tab-index', currentTabIndex);
             }
             catch(exception){ 
             }
         }
     });
</script>
    
</body>
</html>